2025-02-17 15:48:59.896 IST [22632] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-17 15:48:59.956 IST [22632] LOG:  listening on IPv6 address "::1", port 5432
2025-02-17 15:48:59.956 IST [22632] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-17 15:49:00.198 IST [332] LOG:  database system was interrupted; last known up at 2025-02-13 17:29:42 IST
2025-02-17 15:49:06.930 IST [332] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-17 15:49:06.947 IST [332] LOG:  redo starts at 0/3EB7EE8
2025-02-17 15:49:06.947 IST [332] LOG:  invalid record length at 0/3EB7FF0: expected at least 24, got 0
2025-02-17 15:49:06.947 IST [332] LOG:  redo done at 0/3EB7FB8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-17 15:49:06.962 IST [15000] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-17 15:49:07.015 IST [15000] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.019 s, sync=0.012 s, total=0.060 s; sync files=2, longest=0.007 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/3EB7FF0, redo lsn=0/3EB7FF0
2025-02-17 15:49:07.080 IST [22632] LOG:  database system is ready to accept connections
2025-02-17 16:14:06.989 IST [15000] LOG:  checkpoint starting: time
2025-02-17 16:14:07.592 IST [15000] LOG:  checkpoint complete: wrote 6 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.559 s, sync=0.010 s, total=0.604 s; sync files=6, longest=0.006 s, average=0.002 s; distance=3 kB, estimate=3 kB; lsn=0/3EB8F60, redo lsn=0/3EB8F08
2025-02-17 16:19:06.614 IST [15000] LOG:  checkpoint starting: time
2025-02-17 16:19:07.204 IST [15000] LOG:  checkpoint complete: wrote 6 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.550 s, sync=0.011 s, total=0.590 s; sync files=6, longest=0.006 s, average=0.002 s; distance=4 kB, estimate=4 kB; lsn=0/3EBA038, redo lsn=0/3EB9FC8
2025-02-17 16:20:46.984 IST [22632] LOG:  received fast shutdown request
2025-02-17 16:20:46.993 IST [22632] LOG:  aborting any active transactions
2025-02-17 16:20:46.993 IST [19280] ERROR:  canceling statement due to user request
2025-02-17 16:20:47.003 IST [20772] FATAL:  terminating connection due to administrator command
2025-02-17 16:20:47.014 IST [22632] LOG:  background worker "logical replication launcher" (PID 19280) exited with exit code 1
2025-02-17 16:20:47.020 IST [15000] LOG:  shutting down
2025-02-17 16:20:47.028 IST [15000] LOG:  checkpoint starting: shutdown immediate
2025-02-17 16:20:47.063 IST [15000] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.004 s, sync=0.001 s, total=0.043 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=3 kB; lsn=0/3EBA0E8, redo lsn=0/3EBA0E8
2025-02-17 16:20:47.089 IST [22632] LOG:  database system is shut down
2025-02-17 16:22:02.480 IST [23400] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-17 16:22:02.498 IST [23400] LOG:  listening on IPv6 address "::1", port 5432
2025-02-17 16:22:02.498 IST [23400] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-17 16:22:02.884 IST [19696] LOG:  database system was shut down at 2025-02-17 16:20:47 IST
2025-02-17 16:22:02.991 IST [23400] LOG:  database system is ready to accept connections
2025-02-17 16:23:23.230 IST [23400] LOG:  received fast shutdown request
2025-02-17 16:23:23.236 IST [3944] ERROR:  canceling statement due to user request
2025-02-17 16:23:23.238 IST [23400] LOG:  aborting any active transactions
2025-02-17 16:23:23.247 IST [23400] LOG:  background worker "logical replication launcher" (PID 3944) exited with exit code 1
2025-02-17 16:23:23.270 IST [24236] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 16:23:23.270 IST [24236] FATAL:  terminating connection due to administrator command
2025-02-17 16:23:23.270 IST [24236] LOG:  could not send data to client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 16:23:23.283 IST [22780] LOG:  shutting down
2025-02-17 16:23:23.291 IST [22780] LOG:  checkpoint starting: shutdown immediate
2025-02-17 16:23:23.382 IST [22780] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.042 s, sync=0.020 s, total=0.099 s; sync files=7, longest=0.009 s, average=0.003 s; distance=5 kB, estimate=5 kB; lsn=0/3EBB528, redo lsn=0/3EBB528
2025-02-17 16:23:23.409 IST [23400] LOG:  database system is shut down
2025-02-17 16:25:18.414 IST [23428] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-17 16:25:18.421 IST [23428] LOG:  listening on IPv6 address "::1", port 5432
2025-02-17 16:25:18.422 IST [23428] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-17 16:25:18.634 IST [3452] LOG:  database system was shut down at 2025-02-17 16:23:23 IST
2025-02-17 16:25:18.712 IST [23428] LOG:  database system is ready to accept connections
2025-02-17 16:30:18.507 IST [2420] LOG:  checkpoint starting: time
2025-02-17 16:30:18.684 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.136 s, sync=0.017 s, total=0.181 s; sync files=3, longest=0.010 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/3EBB630, redo lsn=0/3EBB5D8
2025-02-17 16:45:18.705 IST [2420] LOG:  checkpoint starting: time
2025-02-17 16:45:18.972 IST [2420] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.228 s, sync=0.014 s, total=0.268 s; sync files=3, longest=0.007 s, average=0.005 s; distance=2 kB, estimate=2 kB; lsn=0/3EBBEF8, redo lsn=0/3EBBEA0
2025-02-17 16:50:18.983 IST [2420] LOG:  checkpoint starting: time
2025-02-17 16:50:19.559 IST [2420] LOG:  checkpoint complete: wrote 6 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.546 s, sync=0.009 s, total=0.577 s; sync files=6, longest=0.006 s, average=0.002 s; distance=5 kB, estimate=5 kB; lsn=0/3EBD3C8, redo lsn=0/3EBD370
2025-02-17 16:55:18.589 IST [2420] LOG:  checkpoint starting: time
2025-02-17 16:55:19.181 IST [2420] LOG:  checkpoint complete: wrote 6 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.558 s, sync=0.008 s, total=0.592 s; sync files=6, longest=0.005 s, average=0.002 s; distance=5 kB, estimate=5 kB; lsn=0/3EBEA80, redo lsn=0/3EBEA28
2025-02-17 17:10:18.180 IST [2420] LOG:  checkpoint starting: time
2025-02-17 17:10:18.770 IST [2420] LOG:  checkpoint complete: wrote 6 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.554 s, sync=0.011 s, total=0.590 s; sync files=6, longest=0.007 s, average=0.002 s; distance=6 kB, estimate=6 kB; lsn=0/3EC0280, redo lsn=0/3EC0228
2025-02-17 17:25:26.366 IST [23456] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 17:25:26.366 IST [21108] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 17:37:08.472 IST [19816] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 17:37:13.082 IST [2420] LOG:  checkpoint starting: immediate force wait
2025-02-17 17:37:13.154 IST [2420] LOG:  checkpoint complete: wrote 5 buffers (0.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.028 s, sync=0.009 s, total=0.072 s; sync files=5, longest=0.007 s, average=0.002 s; distance=4651 kB, estimate=4651 kB; lsn=0/434AE88, redo lsn=0/434AE30
2025-02-17 17:37:16.762 IST [2420] LOG:  checkpoint starting: immediate force wait
2025-02-17 17:37:16.795 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.008 s, sync=0.007 s, total=0.033 s; sync files=4, longest=0.005 s, average=0.002 s; distance=4651 kB, estimate=4651 kB; lsn=0/47D5B80, redo lsn=0/47D5B28
2025-02-17 17:37:20.554 IST [2420] LOG:  checkpoint starting: immediate force wait
2025-02-17 17:37:20.604 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.014 s, sync=0.008 s, total=0.050 s; sync files=4, longest=0.006 s, average=0.002 s; distance=4651 kB, estimate=4651 kB; lsn=0/4C609A8, redo lsn=0/4C60950
2025-02-17 17:37:24.238 IST [2420] LOG:  checkpoint starting: immediate force wait
2025-02-17 17:37:24.278 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.009 s, sync=0.006 s, total=0.041 s; sync files=4, longest=0.005 s, average=0.002 s; distance=4651 kB, estimate=4651 kB; lsn=0/50EB930, redo lsn=0/50EB8D8
2025-02-17 17:37:24.672 IST [20100] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 17:39:35.798 IST [14596] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 17:42:24.276 IST [2420] LOG:  checkpoint starting: time
2025-02-17 17:42:24.633 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.330 s, sync=0.009 s, total=0.358 s; sync files=4, longest=0.006 s, average=0.002 s; distance=2 kB, estimate=4186 kB; lsn=0/50EC490, redo lsn=0/50EC438
2025-02-17 17:46:31.823 IST [15708] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 17:46:46.471 IST [22652] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 17:46:52.283 IST [2420] LOG:  checkpoint starting: immediate force wait
2025-02-17 17:46:52.337 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.021 s, sync=0.008 s, total=0.055 s; sync files=4, longest=0.005 s, average=0.002 s; distance=4652 kB, estimate=4652 kB; lsn=0/5577498, redo lsn=0/5577440
2025-02-17 17:46:56.518 IST [2420] LOG:  checkpoint starting: immediate force wait
2025-02-17 17:46:56.550 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.008 s, sync=0.007 s, total=0.033 s; sync files=4, longest=0.005 s, average=0.002 s; distance=4652 kB, estimate=4652 kB; lsn=0/5A025D8, redo lsn=0/5A02580
2025-02-17 17:47:00.141 IST [2420] LOG:  checkpoint starting: immediate force wait
2025-02-17 17:47:00.178 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.008 s, sync=0.008 s, total=0.038 s; sync files=4, longest=0.006 s, average=0.002 s; distance=4652 kB, estimate=4652 kB; lsn=0/5E8D818, redo lsn=0/5E8D7C0
2025-02-17 17:47:03.525 IST [2420] LOG:  checkpoint starting: immediate force wait
2025-02-17 17:47:03.575 IST [2420] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.012 s, sync=0.007 s, total=0.051 s; sync files=4, longest=0.005 s, average=0.002 s; distance=4652 kB, estimate=4652 kB; lsn=0/6318BB8, redo lsn=0/6318B60
2025-02-17 17:47:04.058 IST [7308] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 17:52:03.585 IST [2420] LOG:  checkpoint starting: time
2025-02-17 17:52:03.724 IST [2420] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.111 s, sync=0.007 s, total=0.140 s; sync files=2, longest=0.006 s, average=0.004 s; distance=0 kB, estimate=4187 kB; lsn=0/6318D50, redo lsn=0/6318CF8
2025-02-17 18:01:43.120 IST [5740] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 18:07:04.895 IST [2712] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-17 18:12:03.723 IST [2420] LOG:  checkpoint starting: time
2025-02-17 18:12:04.643 IST [2420] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.879 s, sync=0.016 s, total=0.923 s; sync files=9, longest=0.006 s, average=0.002 s; distance=7 kB, estimate=3769 kB; lsn=0/631A9D8, redo lsn=0/631A980
2025-02-17 18:17:03.662 IST [2420] LOG:  checkpoint starting: time
2025-02-17 18:17:03.801 IST [2420] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.111 s, sync=0.009 s, total=0.140 s; sync files=2, longest=0.007 s, average=0.005 s; distance=2 kB, estimate=3392 kB; lsn=0/631B530, redo lsn=0/631B4D8
2025-02-17 18:37:03.803 IST [2420] LOG:  checkpoint starting: time
2025-02-17 18:37:05.055 IST [2420] LOG:  checkpoint complete: wrote 12 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.217 s, sync=0.013 s, total=1.253 s; sync files=10, longest=0.007 s, average=0.002 s; distance=7 kB, estimate=3054 kB; lsn=0/631D2C8, redo lsn=0/631D270
2025-02-17 20:31:10.079 IST [25652] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-18 14:02:48.582 IST [26540] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-18 14:02:48.592 IST [26540] LOG:  listening on IPv6 address "::1", port 5432
2025-02-18 14:02:48.593 IST [26540] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-18 14:02:48.758 IST [25836] LOG:  database system was interrupted; last known up at 2025-02-17 18:37:05 IST
2025-02-18 14:02:54.759 IST [25836] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-18 14:02:54.771 IST [25836] LOG:  redo starts at 0/631D270
2025-02-18 14:02:54.771 IST [25836] LOG:  invalid record length at 0/631D378: expected at least 24, got 0
2025-02-18 14:02:54.771 IST [25836] LOG:  redo done at 0/631D340 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-18 14:02:54.783 IST [21404] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-18 14:02:54.840 IST [21404] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.021 s, sync=0.011 s, total=0.064 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/631D378, redo lsn=0/631D378
2025-02-18 14:02:54.920 IST [26540] LOG:  database system is ready to accept connections
2025-02-18 17:30:55.700 IST [10472] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-18 17:30:55.712 IST [10472] LOG:  listening on IPv6 address "::1", port 5432
2025-02-18 17:30:55.712 IST [10472] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-18 17:30:55.874 IST [2156] LOG:  database system was interrupted; last known up at 2025-02-18 14:02:54 IST
2025-02-18 17:31:03.293 IST [2156] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-18 17:31:03.305 IST [2156] LOG:  redo starts at 0/631D3F0
2025-02-18 17:31:03.305 IST [2156] LOG:  invalid record length at 0/631D428: expected at least 24, got 0
2025-02-18 17:31:03.305 IST [2156] LOG:  redo done at 0/631D3F0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-18 17:31:03.319 IST [22912] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-18 17:31:03.372 IST [22912] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.019 s, sync=0.011 s, total=0.060 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/631D428, redo lsn=0/631D428
2025-02-18 17:31:03.452 IST [10472] LOG:  database system is ready to accept connections
2025-02-18 18:37:04.289 IST [10472] LOG:  could not reserve shared memory region (addr=000001A735FF0000) for child 000000000000159C: error code 487
2025-02-18 21:27:21.957 IST [6044] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-20 16:06:46.492 IST [12252] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-20 16:06:46.564 IST [12252] LOG:  listening on IPv6 address "::1", port 5432
2025-02-20 16:06:46.565 IST [12252] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-20 16:06:46.867 IST [23304] LOG:  database system was interrupted; last known up at 2025-02-18 17:31:03 IST
2025-02-20 16:06:54.641 IST [23304] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-20 16:06:54.657 IST [23304] LOG:  redo starts at 0/631D4A0
2025-02-20 16:06:54.657 IST [23304] LOG:  invalid record length at 0/631D4D8: expected at least 24, got 0
2025-02-20 16:06:54.657 IST [23304] LOG:  redo done at 0/631D4A0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-20 16:06:54.674 IST [23112] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-20 16:06:54.733 IST [23112] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.022 s, sync=0.012 s, total=0.068 s; sync files=2, longest=0.007 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/631D4D8, redo lsn=0/631D4D8
2025-02-20 16:06:54.804 IST [12252] LOG:  database system is ready to accept connections
2025-02-20 22:48:06.651 IST [18540] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-20 22:48:06.706 IST [18540] LOG:  listening on IPv6 address "::1", port 5432
2025-02-20 22:48:06.706 IST [18540] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-20 22:48:06.938 IST [19720] LOG:  database system was interrupted; last known up at 2025-02-20 16:06:54 IST
2025-02-20 22:48:12.844 IST [19720] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-20 22:48:12.853 IST [19720] LOG:  redo starts at 0/631D550
2025-02-20 22:48:12.853 IST [19720] LOG:  invalid record length at 0/631D588: expected at least 24, got 0
2025-02-20 22:48:12.853 IST [19720] LOG:  redo done at 0/631D550 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-20 22:48:12.864 IST [12676] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-20 22:48:12.916 IST [12676] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.022 s, sync=0.010 s, total=0.059 s; sync files=2, longest=0.005 s, average=0.005 s; distance=0 kB, estimate=0 kB; lsn=0/631D588, redo lsn=0/631D588
2025-02-20 22:48:12.985 IST [18540] LOG:  database system is ready to accept connections
2025-02-21 02:16:25.045 IST [18540] LOG:  background worker "logical replication launcher" (PID 17792) was terminated by exception 0xC000013A
2025-02-21 02:16:25.045 IST [18540] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-02-21 02:16:25.050 IST [18540] LOG:  terminating any other active server processes
2025-02-21 02:16:25.061 IST [18540] LOG:  received fast shutdown request
^C2025-02-24 08:24:35.819 IST [23956] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-24 08:24:35.873 IST [23956] LOG:  listening on IPv6 address "::1", port 5432
2025-02-24 08:24:35.873 IST [23956] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-24 08:24:36.103 IST [23268] LOG:  database system was interrupted; last known up at 2025-02-20 22:48:12 IST
2025-02-24 08:24:42.161 IST [23268] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-24 08:24:42.170 IST [23268] LOG:  redo starts at 0/631D600
2025-02-24 08:24:42.170 IST [23268] LOG:  invalid record length at 0/631D638: expected at least 24, got 0
2025-02-24 08:24:42.170 IST [23268] LOG:  redo done at 0/631D600 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-24 08:24:42.183 IST [5708] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-24 08:24:42.224 IST [5708] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.015 s, sync=0.009 s, total=0.048 s; sync files=2, longest=0.005 s, average=0.005 s; distance=0 kB, estimate=0 kB; lsn=0/631D638, redo lsn=0/631D638
2025-02-24 08:24:42.283 IST [23956] LOG:  database system is ready to accept connections
2025-02-24 10:49:44.013 IST [5708] LOG:  checkpoint starting: time
2025-02-24 10:49:44.174 IST [5708] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.125 s, sync=0.008 s, total=0.168 s; sync files=2, longest=0.006 s, average=0.004 s; distance=2 kB, estimate=2 kB; lsn=0/631E250, redo lsn=0/631E1F8
2025-02-24 11:34:16.352 IST [10248] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-02-24 12:08:12.345 IST [14876] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-24 12:08:12.398 IST [14876] LOG:  listening on IPv6 address "::1", port 5432
2025-02-24 12:08:12.399 IST [14876] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-24 12:08:12.527 IST [2868] LOG:  database system was interrupted; last known up at 2025-02-24 10:49:44 IST
2025-02-24 12:08:18.372 IST [2868] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-24 12:08:18.383 IST [2868] LOG:  redo starts at 0/631E1F8
2025-02-24 12:08:18.383 IST [2868] LOG:  invalid record length at 0/631E300: expected at least 24, got 0
2025-02-24 12:08:18.383 IST [2868] LOG:  redo done at 0/631E2C8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-24 12:08:18.394 IST [7036] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-24 12:08:18.440 IST [7036] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.018 s, sync=0.010 s, total=0.053 s; sync files=2, longest=0.005 s, average=0.005 s; distance=0 kB, estimate=0 kB; lsn=0/631E300, redo lsn=0/631E300
2025-02-24 12:08:18.504 IST [14876] LOG:  database system is ready to accept connections
2025-02-25 09:17:15.184 IST [10156] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-25 09:17:15.242 IST [10156] LOG:  listening on IPv6 address "::1", port 5432
2025-02-25 09:17:15.242 IST [10156] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-25 09:17:15.375 IST [8540] LOG:  database system was interrupted; last known up at 2025-02-24 12:08:18 IST
2025-02-25 09:17:21.645 IST [8540] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-25 09:17:21.657 IST [8540] LOG:  redo starts at 0/631E378
2025-02-25 09:17:21.657 IST [8540] LOG:  invalid record length at 0/631E3B0: expected at least 24, got 0
2025-02-25 09:17:21.657 IST [8540] LOG:  redo done at 0/631E378 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-25 09:17:21.670 IST [11360] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-25 09:17:21.724 IST [11360] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.020 s, sync=0.011 s, total=0.061 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/631E3B0, redo lsn=0/631E3B0
2025-02-25 09:17:21.794 IST [10156] LOG:  database system is ready to accept connections
2025-02-27 21:20:16.439 IST [27768] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-02-27 21:20:16.581 IST [27768] LOG:  listening on IPv6 address "::1", port 5432
2025-02-27 21:20:16.582 IST [27768] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-27 21:20:16.752 IST [15300] LOG:  database system was interrupted; last known up at 2025-02-25 09:17:21 IST
2025-02-27 21:20:26.154 IST [15300] LOG:  database system was not properly shut down; automatic recovery in progress
2025-02-27 21:20:26.170 IST [15300] LOG:  redo starts at 0/631E428
2025-02-27 21:20:26.170 IST [15300] LOG:  invalid record length at 0/631E460: expected at least 24, got 0
2025-02-27 21:20:26.170 IST [15300] LOG:  redo done at 0/631E428 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-02-27 21:20:26.188 IST [30176] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-02-27 21:20:26.261 IST [30176] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.027 s, sync=0.014 s, total=0.083 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/631E460, redo lsn=0/631E460
2025-02-27 21:20:26.339 IST [27768] LOG:  database system is ready to accept connections
2025-02-27 21:45:26.311 IST [30176] LOG:  checkpoint starting: time
2025-02-27 21:45:27.037 IST [30176] LOG:  checkpoint complete: wrote 7 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.669 s, sync=0.016 s, total=0.741 s; sync files=7, longest=0.007 s, average=0.003 s; distance=7 kB, estimate=7 kB; lsn=0/6320280, redo lsn=0/6320228
2025-03-01 15:40:51.029 IST [22372] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-01 15:40:51.105 IST [22372] LOG:  listening on IPv6 address "::1", port 5432
2025-03-01 15:40:51.105 IST [22372] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-01 15:40:51.290 IST [21512] LOG:  database system was interrupted; last known up at 2025-02-27 21:45:27 IST
2025-03-01 15:41:00.430 IST [21512] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-01 15:41:00.449 IST [21512] LOG:  redo starts at 0/6320228
2025-03-01 15:41:00.450 IST [21512] LOG:  invalid record length at 0/6320330: expected at least 24, got 0
2025-03-01 15:41:00.450 IST [21512] LOG:  redo done at 0/63202F8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-01 15:41:00.470 IST [22468] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-01 15:41:00.568 IST [22468] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.031 s, sync=0.025 s, total=0.109 s; sync files=2, longest=0.015 s, average=0.013 s; distance=0 kB, estimate=0 kB; lsn=0/6320330, redo lsn=0/6320330
2025-03-01 15:41:00.675 IST [22372] LOG:  database system is ready to accept connections
2025-03-01 18:06:26.945 IST [22372] LOG:  autovacuum launcher process (PID 11024) was terminated by exception 0x40010004
2025-03-01 18:06:26.945 IST [22372] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-01 18:06:26.946 IST [22372] LOG:  terminating any other active server processes
2025-03-01 18:06:26.993 IST [22372] LOG:  all server processes terminated; reinitializing
2025-03-01 18:06:27.120 IST [22372] LOG:  startup process (PID 8116) was terminated by exception 0xC000026B
2025-03-01 18:06:27.120 IST [22372] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-01 18:06:27.120 IST [22372] LOG:  aborting startup due to startup process failure
2025-03-01 18:06:27.123 IST [22372] LOG:  database system is shut down
2025-03-02 13:45:27.823 IST [7080] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-02 13:45:27.967 IST [7080] LOG:  listening on IPv6 address "::1", port 5432
2025-03-02 13:45:27.967 IST [7080] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-02 13:45:28.134 IST [23364] LOG:  database system was interrupted; last known up at 2025-03-01 15:41:00 IST
2025-03-02 13:45:38.147 IST [23364] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-02 13:45:38.165 IST [23364] LOG:  redo starts at 0/63203A8
2025-03-02 13:45:38.165 IST [23364] LOG:  invalid record length at 0/63203E0: expected at least 24, got 0
2025-03-02 13:45:38.165 IST [23364] LOG:  redo done at 0/63203A8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-02 13:45:38.182 IST [14732] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-02 13:45:38.245 IST [14732] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.023 s, sync=0.014 s, total=0.073 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/63203E0, redo lsn=0/63203E0
2025-03-02 13:45:38.322 IST [7080] LOG:  database system is ready to accept connections
2025-03-02 14:48:41.948 IST [10784] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-02 14:48:47.343 IST [14732] LOG:  checkpoint starting: immediate force wait
2025-03-02 14:48:47.391 IST [14732] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.017 s, sync=0.009 s, total=0.048 s; sync files=4, longest=0.006 s, average=0.003 s; distance=4653 kB, estimate=4653 kB; lsn=0/67AB838, redo lsn=0/67AB7E0
2025-03-02 14:48:51.687 IST [14732] LOG:  checkpoint starting: immediate force wait
2025-03-02 14:48:51.736 IST [14732] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.012 s, sync=0.009 s, total=0.050 s; sync files=4, longest=0.006 s, average=0.003 s; distance=4653 kB, estimate=4653 kB; lsn=0/6C36DA8, redo lsn=0/6C36D50
2025-03-02 14:48:55.600 IST [14732] LOG:  checkpoint starting: immediate force wait
2025-03-02 14:48:55.656 IST [14732] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.013 s, sync=0.009 s, total=0.056 s; sync files=4, longest=0.006 s, average=0.002 s; distance=4653 kB, estimate=4653 kB; lsn=0/70C2440, redo lsn=0/70C23E8
2025-03-02 14:49:03.850 IST [14732] LOG:  checkpoint starting: immediate force wait
2025-03-02 14:49:03.893 IST [14732] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.012 s, sync=0.008 s, total=0.044 s; sync files=4, longest=0.006 s, average=0.002 s; distance=4653 kB, estimate=4653 kB; lsn=0/754DBE8, redo lsn=0/754DB90
2025-03-02 14:49:04.742 IST [18576] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-02 14:51:02.543 IST [15100] ERROR:  insert or update on table "Account" violates foreign key constraint "Account_userId_fkey"
2025-03-02 14:51:02.543 IST [15100] DETAIL:  Key (userId)=() is not present in table "User".
2025-03-02 14:51:02.543 IST [15100] STATEMENT:  INSERT INTO "public"."Account" ("id","userId","type","provider","providerAccountId") VALUES ($1,$2,$3,$4,$5) RETURNING "public"."Account"."id"
2025-03-02 14:51:29.984 IST [15100] ERROR:  insert or update on table "Account" violates foreign key constraint "Account_userId_fkey"
2025-03-02 14:51:29.984 IST [15100] DETAIL:  Key (userId)=() is not present in table "User".
2025-03-02 14:51:29.984 IST [15100] STATEMENT:  INSERT INTO "public"."Account" ("id","userId","type","provider","providerAccountId") VALUES ($1,$2,$3,$4,$5) RETURNING "public"."Account"."id"
2025-03-02 14:54:03.920 IST [14732] LOG:  checkpoint starting: time
2025-03-02 14:54:10.982 IST [14732] LOG:  checkpoint complete: wrote 62 buffers (0.4%); 0 WAL file(s) added, 0 removed, 0 recycled; write=6.977 s, sync=0.060 s, total=7.062 s; sync files=55, longest=0.007 s, average=0.002 s; distance=204 kB, estimate=4208 kB; lsn=0/7580E48, redo lsn=0/7580DB8
2025-03-02 14:59:03.221 IST [14732] LOG:  checkpoint starting: time
2025-03-02 14:59:04.160 IST [14732] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.885 s, sync=0.013 s, total=0.940 s; sync files=9, longest=0.007 s, average=0.002 s; distance=9 kB, estimate=3788 kB; lsn=0/7583278, redo lsn=0/7583220
2025-03-02 18:20:29.832 IST [14732] LOG:  checkpoint starting: time
2025-03-02 18:20:31.199 IST [14732] LOG:  checkpoint complete: wrote 12 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.308 s, sync=0.015 s, total=1.368 s; sync files=12, longest=0.008 s, average=0.002 s; distance=11 kB, estimate=3411 kB; lsn=0/7585E88, redo lsn=0/7585E30
2025-03-03 10:41:03.211 IST [21188] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-03 10:41:03.284 IST [21188] LOG:  listening on IPv6 address "::1", port 5432
2025-03-03 10:41:03.284 IST [21188] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-03 10:41:03.440 IST [21524] LOG:  database system was interrupted; last known up at 2025-03-02 18:20:31 IST
2025-03-03 10:41:11.114 IST [21524] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-03 10:41:11.130 IST [21524] LOG:  redo starts at 0/7585E30
2025-03-03 10:41:11.130 IST [21524] LOG:  invalid record length at 0/7585F38: expected at least 24, got 0
2025-03-03 10:41:11.130 IST [21524] LOG:  redo done at 0/7585F00 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-03 10:41:11.148 IST [17884] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-03 10:41:11.213 IST [17884] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.023 s, sync=0.014 s, total=0.074 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/7585F38, redo lsn=0/7585F38
2025-03-03 10:41:11.286 IST [21188] LOG:  database system is ready to accept connections
2025-03-03 13:48:30.859 IST [21188] LOG:  background worker "logical replication launcher" (PID 11552) was terminated by exception 0xC000013A
2025-03-03 13:48:30.859 IST [21188] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-03 13:48:30.861 IST [21188] LOG:  terminating any other active server processes
2025-03-03 13:48:30.883 IST [21188] LOG:  all server processes terminated; reinitializing
^C2025-03-04 08:29:05.858 IST [3320] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-04 08:29:05.932 IST [3320] LOG:  listening on IPv6 address "::1", port 5432
2025-03-04 08:29:05.932 IST [3320] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-04 08:29:06.363 IST [15268] LOG:  database system was interrupted; last known up at 2025-03-03 10:41:11 IST
2025-03-04 08:29:16.384 IST [15268] LOG:  syncing data directory (fsync), elapsed time: 10.01 s, current path: ./base/5/2654
2025-03-04 08:29:18.312 IST [15268] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-04 08:29:18.331 IST [15268] LOG:  redo starts at 0/7585FB0
2025-03-04 08:29:18.332 IST [15268] LOG:  invalid record length at 0/7585FE8: expected at least 24, got 0
2025-03-04 08:29:18.332 IST [15268] LOG:  redo done at 0/7585FB0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-04 08:29:18.353 IST [10532] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-04 08:29:18.461 IST [10532] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.028 s, sync=0.037 s, total=0.120 s; sync files=2, longest=0.026 s, average=0.019 s; distance=0 kB, estimate=0 kB; lsn=0/7585FE8, redo lsn=0/7585FE8
2025-03-04 08:29:18.556 IST [3320] LOG:  database system is ready to accept connections
2025-03-04 08:39:18.481 IST [10532] LOG:  checkpoint starting: time
2025-03-04 08:39:19.314 IST [10532] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.779 s, sync=0.018 s, total=0.834 s; sync files=8, longest=0.011 s, average=0.003 s; distance=9 kB, estimate=9 kB; lsn=0/75886D8, redo lsn=0/7588680
2025-03-04 08:39:46.845 IST [22004] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-04 08:39:54.267 IST [3320] LOG:  received fast shutdown request
2025-03-04 08:39:54.272 IST [21396] ERROR:  canceling statement due to user request
2025-03-04 08:39:54.274 IST [3320] LOG:  aborting any active transactions
2025-03-04 08:39:54.280 IST [3320] LOG:  background worker "logical replication launcher" (PID 21396) exited with exit code 1
2025-03-04 08:39:54.280 IST [10532] LOG:  shutting down
2025-03-04 08:39:54.290 IST [10532] LOG:  checkpoint starting: shutdown immediate
2025-03-04 08:39:54.341 IST [10532] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.005 s, sync=0.001 s, total=0.061 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=8 kB; lsn=0/7588788, redo lsn=0/7588788
2025-03-04 08:39:54.374 IST [3320] LOG:  database system is shut down
2025-03-04 09:33:07.853 IST [2940] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-04 09:33:07.950 IST [2940] LOG:  listening on IPv6 address "::1", port 5432
2025-03-04 09:33:07.950 IST [2940] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-04 09:33:08.274 IST [13852] LOG:  database system was shut down at 2025-03-04 08:39:54 IST
2025-03-04 09:33:08.480 IST [2940] LOG:  database system is ready to accept connections
2025-03-04 09:38:08.236 IST [4412] LOG:  checkpoint starting: time
2025-03-04 09:38:08.308 IST [4412] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.030 s, sync=0.015 s, total=0.088 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/7588890, redo lsn=0/7588838
2025-03-04 11:14:03.327 IST [2940] LOG:  background worker "logical replication launcher" (PID 23476) was terminated by exception 0x40010004
2025-03-04 11:14:03.327 IST [2940] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-04 11:14:03.329 IST [2940] LOG:  terminating any other active server processes
2025-03-04 11:14:03.445 IST [2940] LOG:  all server processes terminated; reinitializing
^C2025-03-04 21:15:52.495 IST [14908] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-04 21:15:52.550 IST [14908] LOG:  listening on IPv6 address "::1", port 5432
2025-03-04 21:15:52.550 IST [14908] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-04 21:15:52.694 IST [9800] LOG:  database system was interrupted; last known up at 2025-03-04 09:38:08 IST
2025-03-04 21:15:59.764 IST [9800] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-04 21:15:59.777 IST [9800] LOG:  redo starts at 0/7588838
2025-03-04 21:15:59.777 IST [9800] LOG:  invalid record length at 0/7588940: expected at least 24, got 0
2025-03-04 21:15:59.777 IST [9800] LOG:  redo done at 0/7588908 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-04 21:15:59.789 IST [21632] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-04 21:15:59.840 IST [21632] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.019 s, sync=0.011 s, total=0.058 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/7588940, redo lsn=0/7588940
2025-03-04 21:15:59.913 IST [14908] LOG:  database system is ready to accept connections
2025-03-04 21:24:48.938 IST [17808] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-04 21:24:53.255 IST [21632] LOG:  checkpoint starting: immediate force wait
2025-03-04 21:24:53.291 IST [21632] LOG:  checkpoint complete: wrote 5 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.012 s, sync=0.007 s, total=0.037 s; sync files=5, longest=0.004 s, average=0.002 s; distance=4697 kB, estimate=4697 kB; lsn=0/7A1EFE8, redo lsn=0/7A1EF90
2025-03-04 21:24:56.725 IST [21632] LOG:  checkpoint starting: immediate force wait
2025-03-04 21:24:56.769 IST [21632] LOG:  checkpoint complete: wrote 10 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.011 s, sync=0.010 s, total=0.044 s; sync files=9, longest=0.005 s, average=0.002 s; distance=4697 kB, estimate=4697 kB; lsn=0/7EB5790, redo lsn=0/7EB5738
2025-03-04 21:24:59.580 IST [21632] LOG:  checkpoint starting: immediate force wait
2025-03-04 21:24:59.626 IST [21632] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.010 s, sync=0.007 s, total=0.047 s; sync files=4, longest=0.005 s, average=0.002 s; distance=4698 kB, estimate=4698 kB; lsn=0/834C078, redo lsn=0/834C020
2025-03-04 21:25:06.081 IST [21632] LOG:  checkpoint starting: immediate force wait
2025-03-04 21:25:06.122 IST [21632] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.010 s, sync=0.008 s, total=0.042 s; sync files=4, longest=0.006 s, average=0.002 s; distance=4698 kB, estimate=4698 kB; lsn=0/87E2A70, redo lsn=0/87E2A18
2025-03-04 21:25:06.906 IST [22720] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-04 21:30:06.140 IST [21632] LOG:  checkpoint starting: time
2025-03-04 21:30:14.232 IST [21632] LOG:  checkpoint complete: wrote 73 buffers (0.4%); 0 WAL file(s) added, 0 removed, 0 recycled; write=7.933 s, sync=0.128 s, total=8.093 s; sync files=66, longest=0.009 s, average=0.002 s; distance=346 kB, estimate=4263 kB; lsn=0/8839610, redo lsn=0/88395B8
2025-03-04 21:45:11.607 IST [7540] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-05 10:07:43.265 IST [20348] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-05 10:07:43.339 IST [20348] LOG:  listening on IPv6 address "::1", port 5432
2025-03-05 10:07:43.339 IST [20348] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-05 10:07:43.551 IST [20944] LOG:  database system was interrupted; last known up at 2025-03-04 21:30:14 IST
2025-03-05 10:07:51.949 IST [20944] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-05 10:07:51.968 IST [20944] LOG:  redo starts at 0/88395B8
2025-03-05 10:07:51.968 IST [20944] LOG:  invalid record length at 0/88396C0: expected at least 24, got 0
2025-03-05 10:07:51.968 IST [20944] LOG:  redo done at 0/8839688 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-05 10:07:51.986 IST [19528] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-05 10:07:52.052 IST [19528] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.023 s, sync=0.014 s, total=0.075 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/88396C0, redo lsn=0/88396C0
2025-03-05 10:07:52.126 IST [20348] LOG:  database system is ready to accept connections
2025-03-05 11:04:08.423 IST [21800] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-10 10:45:20.440 IST [11976] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-10 10:45:20.655 IST [11976] LOG:  listening on IPv6 address "::1", port 5432
2025-03-10 10:45:20.655 IST [11976] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-10 10:45:21.456 IST [11412] LOG:  database system was interrupted; last known up at 2025-03-05 10:07:52 IST
2025-03-10 10:45:31.523 IST [11412] LOG:  syncing data directory (fsync), elapsed time: 10.03 s, current path: ./base/16387/6117
2025-03-10 10:45:38.931 IST [11412] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-10 10:45:38.961 IST [11412] LOG:  redo starts at 0/8839738
2025-03-10 10:45:38.961 IST [11412] LOG:  invalid record length at 0/8839770: expected at least 24, got 0
2025-03-10 10:45:38.962 IST [11412] LOG:  redo done at 0/8839738 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-10 10:45:38.986 IST [20320] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-10 10:45:39.105 IST [20320] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.039 s, sync=0.022 s, total=0.133 s; sync files=2, longest=0.013 s, average=0.011 s; distance=0 kB, estimate=0 kB; lsn=0/8839770, redo lsn=0/8839770
2025-03-10 10:45:39.253 IST [11976] LOG:  database system is ready to accept connections
2025-03-10 15:04:13.245 IST [15852] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-10 15:04:13.263 IST [15852] LOG:  listening on IPv6 address "::1", port 5432
2025-03-10 15:04:13.264 IST [15852] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-10 15:04:13.440 IST [21784] LOG:  database system was interrupted; last known up at 2025-03-10 10:45:39 IST
2025-03-10 15:04:20.292 IST [21784] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-10 15:04:20.307 IST [21784] LOG:  redo starts at 0/88397E8
2025-03-10 15:04:20.307 IST [21784] LOG:  invalid record length at 0/8839820: expected at least 24, got 0
2025-03-10 15:04:20.308 IST [21784] LOG:  redo done at 0/88397E8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-10 15:04:20.321 IST [18204] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-10 15:04:20.371 IST [18204] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.017 s, sync=0.012 s, total=0.057 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/8839820, redo lsn=0/8839820
2025-03-10 15:04:20.440 IST [15852] LOG:  database system is ready to accept connections
2025-03-10 15:04:25.898 IST [15852] LOG:  background worker "logical replication launcher" (PID 17892) was terminated by exception 0xC000013A
2025-03-10 15:04:25.898 IST [15852] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-10 15:04:25.898 IST [15852] LOG:  terminating any other active server processes
2025-03-10 15:04:25.924 IST [15852] LOG:  all server processes terminated; reinitializing
2025-03-10 15:04:26.036 IST [15852] LOG:  startup process (PID 14644) was terminated by exception 0xC0000142
2025-03-10 15:04:26.036 IST [15852] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-10 15:04:26.036 IST [15852] LOG:  aborting startup due to startup process failure
2025-03-10 15:04:26.040 IST [15852] LOG:  database system is shut down
2025-03-10 15:43:01.746 IST [16284] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-10 15:43:01.758 IST [16284] LOG:  listening on IPv6 address "::1", port 5432
2025-03-10 15:43:01.759 IST [16284] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-10 15:43:01.936 IST [2388] LOG:  database system was interrupted; last known up at 2025-03-10 15:04:20 IST
2025-03-10 15:43:07.733 IST [2388] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-10 15:43:07.746 IST [2388] LOG:  invalid record length at 0/8839898: expected at least 24, got 0
2025-03-10 15:43:07.746 IST [2388] LOG:  redo is not required
2025-03-10 15:43:07.763 IST [18700] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-10 15:43:07.815 IST [18700] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.018 s, sync=0.011 s, total=0.061 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/8839898, redo lsn=0/8839898
2025-03-10 15:43:07.892 IST [16284] LOG:  database system is ready to accept connections
2025-03-11 18:39:42.425 IST [8252] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-11 18:39:42.495 IST [8252] LOG:  listening on IPv6 address "::1", port 5432
2025-03-11 18:39:42.495 IST [8252] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-11 18:39:42.961 IST [20228] LOG:  database system was interrupted; last known up at 2025-03-10 15:43:07 IST
2025-03-11 18:39:52.980 IST [20228] LOG:  syncing data directory (fsync), elapsed time: 10.01 s, current path: ./base/5/3601_fsm
2025-03-11 18:39:54.244 IST [20228] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-11 18:39:54.264 IST [20228] LOG:  redo starts at 0/8839910
2025-03-11 18:39:54.264 IST [20228] LOG:  invalid record length at 0/8839948: expected at least 24, got 0
2025-03-11 18:39:54.264 IST [20228] LOG:  redo done at 0/8839910 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-11 18:39:54.286 IST [23380] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-11 18:39:54.368 IST [23380] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.028 s, sync=0.017 s, total=0.094 s; sync files=2, longest=0.009 s, average=0.009 s; distance=0 kB, estimate=0 kB; lsn=0/8839948, redo lsn=0/8839948
2025-03-11 18:39:54.493 IST [8252] LOG:  database system is ready to accept connections
2025-03-11 19:43:08.816 IST [8252] LOG:  server process (PID 10376) was terminated by exception 0x40010004
2025-03-11 19:43:08.816 IST [8252] DETAIL:  Failed process was running: SELECT "public"."User"."id", "public"."User"."name", "public"."User"."username", "public"."User"."email", "public"."User"."emailVerified", "public"."User"."passwordHash", "public"."User"."image" FROM "public"."User" WHERE "public"."User"."id" IN ($1) OFFSET $2
2025-03-11 19:43:08.816 IST [8252] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-11 19:43:08.817 IST [8252] LOG:  terminating any other active server processes
2025-03-11 19:43:08.856 IST [8252] LOG:  all server processes terminated; reinitializing
2025-03-11 19:43:09.068 IST [8252] LOG:  startup process (PID 20500) was terminated by exception 0xC0000142
2025-03-11 19:43:09.068 IST [8252] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-11 19:43:09.068 IST [8252] LOG:  aborting startup due to startup process failure
2025-03-11 19:43:09.079 IST [8252] LOG:  database system is shut down
2025-03-13 05:33:30.697 IST [22884] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-13 05:33:30.761 IST [22884] LOG:  listening on IPv6 address "::1", port 5432
2025-03-13 05:33:30.761 IST [22884] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-13 05:33:30.965 IST [17312] LOG:  database system was interrupted; last known up at 2025-03-11 18:39:54 IST
2025-03-13 05:33:40.298 IST [17312] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-13 05:33:40.317 IST [17312] LOG:  redo starts at 0/88399C0
2025-03-13 05:33:40.318 IST [17312] LOG:  invalid record length at 0/88399F8: expected at least 24, got 0
2025-03-13 05:33:40.318 IST [17312] LOG:  redo done at 0/88399C0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-13 05:33:40.348 IST [17972] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-13 05:33:40.421 IST [17972] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.026 s, sync=0.014 s, total=0.087 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/88399F8, redo lsn=0/88399F8
2025-03-13 05:33:40.524 IST [22884] LOG:  database system is ready to accept connections
2025-03-13 06:49:16.219 IST [9340] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-13 17:42:58.289 IST [23376] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-13 17:42:58.357 IST [23376] LOG:  listening on IPv6 address "::1", port 5432
2025-03-13 17:42:58.358 IST [23376] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-13 17:42:58.517 IST [10076] LOG:  database system was interrupted; last known up at 2025-03-13 05:33:40 IST
2025-03-13 17:43:06.479 IST [10076] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-13 17:43:06.496 IST [10076] LOG:  redo starts at 0/8839A70
2025-03-13 17:43:06.496 IST [10076] LOG:  invalid record length at 0/8839AA8: expected at least 24, got 0
2025-03-13 17:43:06.496 IST [10076] LOG:  redo done at 0/8839A70 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-13 17:43:06.514 IST [21588] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-13 17:43:06.584 IST [21588] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.025 s, sync=0.016 s, total=0.079 s; sync files=2, longest=0.009 s, average=0.008 s; distance=0 kB, estimate=0 kB; lsn=0/8839AA8, redo lsn=0/8839AA8
2025-03-13 17:43:06.665 IST [23376] LOG:  database system is ready to accept connections
2025-03-14 08:11:46.294 IST [23376] LOG:  background worker "logical replication launcher" (PID 8288) was terminated by exception 0x40010004
2025-03-14 08:11:46.294 IST [23376] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-14 08:11:46.297 IST [23376] LOG:  terminating any other active server processes
2025-03-14 08:11:46.404 IST [23376] LOG:  all server processes terminated; reinitializing
2025-03-14 08:11:47.500 IST [23376] LOG:  startup process (PID 22156) was terminated by exception 0xC000026B
2025-03-14 08:11:47.500 IST [23376] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-14 08:11:47.500 IST [23376] LOG:  aborting startup due to startup process failure
2025-03-14 08:11:47.506 IST [23376] LOG:  database system is shut down
2025-03-14 18:57:05.699 IST [7568] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-14 18:57:05.896 IST [7568] LOG:  listening on IPv6 address "::1", port 5432
2025-03-14 18:57:05.897 IST [7568] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-14 18:57:06.451 IST [22632] LOG:  database system was interrupted; last known up at 2025-03-13 17:43:06 IST
2025-03-14 18:57:16.514 IST [22632] LOG:  syncing data directory (fsync), elapsed time: 10.04 s, current path: ./base/1/3394_vm
2025-03-14 18:57:26.494 IST [22632] LOG:  syncing data directory (fsync), elapsed time: 20.02 s, current path: ./base/4/2618_fsm
2025-03-14 18:57:33.071 IST [22632] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-14 18:57:33.246 IST [22632] LOG:  redo starts at 0/8839B20
2025-03-14 18:57:33.247 IST [22632] LOG:  invalid record length at 0/8839B58: expected at least 24, got 0
2025-03-14 18:57:33.250 IST [22632] LOG:  redo done at 0/8839B20 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.01 s
2025-03-14 18:57:33.316 IST [22604] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-14 18:57:33.580 IST [22604] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.139 s, sync=0.035 s, total=0.297 s; sync files=2, longest=0.018 s, average=0.018 s; distance=0 kB, estimate=0 kB; lsn=0/8839B58, redo lsn=0/8839B58
2025-03-14 18:57:33.782 IST [7568] LOG:  database system is ready to accept connections
2025-03-14 19:31:09.204 IST [7568] LOG:  autovacuum launcher process (PID 23308) was terminated by exception 0x40010004
2025-03-14 19:31:09.204 IST [7568] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-14 19:31:09.205 IST [7568] LOG:  terminating any other active server processes
^C2025-03-15 15:17:07.277 IST [22428] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-15 15:17:07.345 IST [22428] LOG:  listening on IPv6 address "::1", port 5432
2025-03-15 15:17:07.346 IST [22428] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-15 15:17:07.536 IST [8560] LOG:  database system was interrupted; last known up at 2025-03-14 18:57:33 IST
2025-03-15 15:17:15.548 IST [8560] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-15 15:17:15.563 IST [8560] LOG:  redo starts at 0/8839BD0
2025-03-15 15:17:15.563 IST [8560] LOG:  invalid record length at 0/8839C08: expected at least 24, got 0
2025-03-15 15:17:15.563 IST [8560] LOG:  redo done at 0/8839BD0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-15 15:17:15.580 IST [22232] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-15 15:17:15.650 IST [22232] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.026 s, sync=0.015 s, total=0.078 s; sync files=2, longest=0.008 s, average=0.008 s; distance=0 kB, estimate=0 kB; lsn=0/8839C08, redo lsn=0/8839C08
2025-03-15 15:17:15.727 IST [22428] LOG:  database system is ready to accept connections
2025-03-15 19:51:05.876 IST [20540] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-15 19:54:12.781 IST [17532] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-15 19:54:13.451 IST [22384] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-15 19:57:10.858 IST [22232] LOG:  checkpoint starting: time
2025-03-15 19:57:12.629 IST [22232] LOG:  checkpoint complete: wrote 16 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.737 s, sync=0.008 s, total=1.775 s; sync files=11, longest=0.002 s, average=0.001 s; distance=35 kB, estimate=35 kB; lsn=0/8842A00, redo lsn=0/88429A8
2025-03-15 20:02:27.312 IST [16412] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-15 20:02:39.321 IST [10104] LOG:  could not receive data from client: An established connection was aborted by the software in your host machine.
	
2025-03-15 20:02:39.757 IST [4324] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-15 20:25:11.347 IST [23336] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-15 20:27:10.732 IST [22232] LOG:  checkpoint starting: time
2025-03-15 20:27:12.078 IST [22232] LOG:  checkpoint complete: wrote 13 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.310 s, sync=0.011 s, total=1.348 s; sync files=9, longest=0.006 s, average=0.002 s; distance=2 kB, estimate=32 kB; lsn=0/8843510, redo lsn=0/88434B8
2025-03-16 11:46:25.199 IST [14020] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-16 11:46:25.267 IST [14020] LOG:  listening on IPv6 address "::1", port 5432
2025-03-16 11:46:25.267 IST [14020] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-16 11:46:25.438 IST [8184] LOG:  database system was interrupted; last known up at 2025-03-15 20:27:12 IST
2025-03-16 11:46:35.460 IST [8184] LOG:  syncing data directory (fsync), elapsed time: 10.01 s, current path: ./base/4/2686
2025-03-16 11:46:39.220 IST [8184] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-16 11:46:39.236 IST [8184] LOG:  redo starts at 0/88434B8
2025-03-16 11:46:39.237 IST [8184] LOG:  invalid record length at 0/88435C0: expected at least 24, got 0
2025-03-16 11:46:39.237 IST [8184] LOG:  redo done at 0/8843588 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-16 11:46:39.253 IST [12388] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-16 11:46:39.314 IST [12388] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.022 s, sync=0.013 s, total=0.069 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/88435C0, redo lsn=0/88435C0
2025-03-16 11:46:39.401 IST [14020] LOG:  database system is ready to accept connections
2025-03-16 12:11:39.350 IST [12388] LOG:  checkpoint starting: time
2025-03-16 12:11:40.543 IST [12388] LOG:  checkpoint complete: wrote 11 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.128 s, sync=0.025 s, total=1.203 s; sync files=11, longest=0.012 s, average=0.003 s; distance=3 kB, estimate=3 kB; lsn=0/8844598, redo lsn=0/8844540
2025-03-16 12:21:39.614 IST [12388] LOG:  checkpoint starting: time
2025-03-16 12:21:39.811 IST [12388] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.145 s, sync=0.019 s, total=0.198 s; sync files=2, longest=0.011 s, average=0.010 s; distance=3 kB, estimate=3 kB; lsn=0/8845428, redo lsn=0/88453D0
2025-03-16 12:28:35.615 IST [5492] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-16 12:41:39.874 IST [12388] LOG:  checkpoint starting: time
2025-03-16 12:41:41.252 IST [12388] LOG:  checkpoint complete: wrote 13 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.320 s, sync=0.026 s, total=1.392 s; sync files=13, longest=0.012 s, average=0.002 s; distance=12 kB, estimate=12 kB; lsn=0/8848618, redo lsn=0/88485C0
2025-03-16 13:00:59.583 IST [9368] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-16 13:21:39.397 IST [12388] LOG:  checkpoint starting: time
2025-03-16 13:21:39.561 IST [12388] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.124 s, sync=0.011 s, total=0.178 s; sync files=2, longest=0.009 s, average=0.006 s; distance=3 kB, estimate=11 kB; lsn=0/8849548, redo lsn=0/88494F0
2025-03-16 14:02:49.679 IST [22464] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-18 11:27:26.159 IST [2780] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-18 11:27:26.172 IST [2780] LOG:  listening on IPv6 address "::1", port 5432
2025-03-18 11:27:26.172 IST [2780] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-18 11:27:26.325 IST [7008] LOG:  database system was interrupted; last known up at 2025-03-18 11:23:31 IST
2025-03-18 11:27:33.076 IST [7008] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-18 11:27:33.092 IST [7008] LOG:  redo starts at 0/886BB58
2025-03-18 11:27:33.092 IST [7008] LOG:  invalid record length at 0/886BB90: expected at least 24, got 0
2025-03-18 11:27:33.092 IST [7008] LOG:  redo done at 0/886BB58 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-18 11:27:33.111 IST [14260] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-18 11:27:33.188 IST [14260] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.027 s, sync=0.015 s, total=0.087 s; sync files=2, longest=0.009 s, average=0.008 s; distance=0 kB, estimate=0 kB; lsn=0/886BB90, redo lsn=0/886BB90
2025-03-18 11:27:33.265 IST [2780] LOG:  database system is ready to accept connections
2025-03-18 11:32:33.196 IST [14260] LOG:  checkpoint starting: time
2025-03-18 11:32:33.917 IST [14260] LOG:  checkpoint complete: wrote 7 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.661 s, sync=0.016 s, total=0.721 s; sync files=7, longest=0.009 s, average=0.003 s; distance=16 kB, estimate=16 kB; lsn=0/886FF10, redo lsn=0/886FEB8
2025-03-18 14:54:35.381 IST [14260] LOG:  checkpoint starting: time
2025-03-18 14:54:35.543 IST [14260] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.126 s, sync=0.008 s, total=0.168 s; sync files=2, longest=0.006 s, average=0.004 s; distance=5 kB, estimate=15 kB; lsn=0/8871608, redo lsn=0/88715B0
2025-03-18 16:54:35.496 IST [14260] LOG:  checkpoint starting: time
2025-03-18 16:54:36.871 IST [14260] LOG:  checkpoint complete: wrote 13 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.323 s, sync=0.021 s, total=1.379 s; sync files=13, longest=0.007 s, average=0.002 s; distance=19 kB, estimate=19 kB; lsn=0/8876408, redo lsn=0/88763B0
2025-03-18 21:55:40.578 IST [2780] LOG:  background worker "logical replication launcher" (PID 20908) was terminated by exception 0x40010004
2025-03-18 21:55:40.578 IST [2780] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-18 21:55:40.580 IST [2780] LOG:  terminating any other active server processes
2025-03-18 21:55:40.739 IST [2780] LOG:  all server processes terminated; reinitializing
2025-03-18 21:55:41.392 IST [2780] LOG:  startup process (PID 28396) was terminated by exception 0xC000026B
2025-03-18 21:55:41.392 IST [2780] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-18 21:55:41.392 IST [2780] LOG:  aborting startup due to startup process failure
2025-03-18 21:55:41.398 IST [2780] LOG:  database system is shut down
2025-03-19 10:29:17.070 IST [11188] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-19 10:29:17.143 IST [11188] LOG:  listening on IPv6 address "::1", port 5432
2025-03-19 10:29:17.143 IST [11188] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-19 10:29:17.317 IST [18104] LOG:  database system was interrupted; last known up at 2025-03-18 16:54:36 IST
2025-03-19 10:29:25.892 IST [18104] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-19 10:29:25.917 IST [18104] LOG:  redo starts at 0/88763B0
2025-03-19 10:29:25.918 IST [18104] LOG:  invalid record length at 0/88764B8: expected at least 24, got 0
2025-03-19 10:29:25.918 IST [18104] LOG:  redo done at 0/8876480 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-19 10:29:25.939 IST [20816] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-19 10:29:26.043 IST [20816] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.038 s, sync=0.019 s, total=0.116 s; sync files=2, longest=0.010 s, average=0.010 s; distance=0 kB, estimate=0 kB; lsn=0/88764B8, redo lsn=0/88764B8
2025-03-19 10:29:26.147 IST [11188] LOG:  database system is ready to accept connections
2025-03-19 16:54:51.316 IST [15748] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-19 16:54:51.370 IST [15748] LOG:  listening on IPv6 address "::1", port 5432
2025-03-19 16:54:51.370 IST [15748] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-19 16:54:51.589 IST [15704] LOG:  database system was interrupted; last known up at 2025-03-19 10:29:26 IST
2025-03-19 16:54:57.654 IST [15704] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-19 16:54:57.666 IST [15704] LOG:  redo starts at 0/8876530
2025-03-19 16:54:57.666 IST [15704] LOG:  invalid record length at 0/8876568: expected at least 24, got 0
2025-03-19 16:54:57.666 IST [15704] LOG:  redo done at 0/8876530 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-19 16:54:57.681 IST [18368] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-19 16:54:57.740 IST [18368] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.019 s, sync=0.014 s, total=0.067 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/8876568, redo lsn=0/8876568
2025-03-19 16:54:57.795 IST [15748] LOG:  database system is ready to accept connections
2025-03-19 17:04:57.749 IST [18368] LOG:  checkpoint starting: time
2025-03-19 17:04:58.560 IST [18368] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.770 s, sync=0.016 s, total=0.813 s; sync files=8, longest=0.006 s, average=0.002 s; distance=4 kB, estimate=4 kB; lsn=0/8877718, redo lsn=0/88776C0
2025-03-19 21:46:40.914 IST [23788] FATAL:  terminating connection due to unexpected postmaster exit
2025-03-20 09:44:26.879 IST [22112] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-20 09:44:26.896 IST [22112] LOG:  listening on IPv6 address "::1", port 5432
2025-03-20 09:44:26.896 IST [22112] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-20 09:44:27.115 IST [17056] LOG:  database system was interrupted; last known up at 2025-03-19 17:04:58 IST
2025-03-20 09:44:36.243 IST [17056] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-20 09:44:36.262 IST [17056] LOG:  redo starts at 0/88776C0
2025-03-20 09:44:36.262 IST [17056] LOG:  invalid record length at 0/88777C8: expected at least 24, got 0
2025-03-20 09:44:36.262 IST [17056] LOG:  redo done at 0/8877790 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-20 09:44:36.280 IST [22480] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-20 09:44:36.347 IST [22480] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.025 s, sync=0.014 s, total=0.077 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/88777C8, redo lsn=0/88777C8
2025-03-20 09:44:36.518 IST [22112] LOG:  database system is ready to accept connections
2025-03-20 10:35:10.829 IST [22480] LOG:  checkpoint starting: time
2025-03-20 10:35:11.654 IST [22480] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.771 s, sync=0.024 s, total=0.837 s; sync files=8, longest=0.012 s, average=0.003 s; distance=4 kB, estimate=4 kB; lsn=0/8878B00, redo lsn=0/8878AA8
2025-03-20 10:50:10.704 IST [22480] LOG:  checkpoint starting: time
2025-03-20 10:50:11.619 IST [22480] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.856 s, sync=0.019 s, total=0.916 s; sync files=8, longest=0.008 s, average=0.003 s; distance=5 kB, estimate=5 kB; lsn=0/8879F60, redo lsn=0/8879F08
2025-03-20 11:02:45.687 IST [10732] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-20 17:02:12.475 IST [22112] LOG:  received fast shutdown request
2025-03-20 17:02:12.480 IST [6680] ERROR:  canceling statement due to user request
2025-03-20 17:02:12.481 IST [22112] LOG:  aborting any active transactions
2025-03-20 17:02:12.487 IST [25144] FATAL:  terminating connection due to administrator command
2025-03-20 17:02:12.496 IST [22112] LOG:  background worker "logical replication launcher" (PID 6680) exited with exit code 1
2025-03-20 17:02:12.505 IST [22480] LOG:  shutting down
2025-03-20 17:02:12.513 IST [22480] LOG:  checkpoint starting: shutdown immediate
2025-03-20 17:02:12.540 IST [22480] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.004 s, sync=0.001 s, total=0.035 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=4 kB; lsn=0/887A028, redo lsn=0/887A028
2025-03-20 17:02:12.558 IST [22112] LOG:  database system is shut down
2025-03-20 17:02:42.284 IST [22716] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-20 17:02:42.292 IST [22716] LOG:  listening on IPv6 address "::1", port 5432
2025-03-20 17:02:42.293 IST [22716] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-20 17:02:42.433 IST [22288] LOG:  database system was shut down at 2025-03-20 17:02:12 IST
2025-03-20 17:02:42.537 IST [22716] LOG:  database system is ready to accept connections
2025-03-20 17:02:55.368 IST [23684] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-20 17:02:55.946 IST [2728] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-20 17:03:18.221 IST [18780] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-20 17:03:22.780 IST [22736] LOG:  checkpoint starting: immediate force wait
2025-03-20 17:03:22.859 IST [22736] LOG:  checkpoint complete: wrote 16 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.033 s, sync=0.021 s, total=0.082 s; sync files=14, longest=0.006 s, average=0.002 s; distance=4894 kB, estimate=4894 kB; lsn=0/8D41A78, redo lsn=0/8D41A20
2025-03-20 17:03:27.030 IST [22736] LOG:  checkpoint starting: immediate force wait
2025-03-20 17:03:27.098 IST [22736] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.018 s, sync=0.010 s, total=0.069 s; sync files=7, longest=0.006 s, average=0.002 s; distance=4851 kB, estimate=4890 kB; lsn=0/91FE9E8, redo lsn=0/91FE990
2025-03-20 17:03:48.847 IST [20052] ERROR:  table "_prisma_migrations" does not exist
2025-03-20 17:03:48.847 IST [20052] STATEMENT:  DROP TABLE _prisma_migrations
2025-03-20 17:03:52.280 IST [22736] LOG:  checkpoint starting: immediate force wait
2025-03-20 17:03:52.554 IST [22736] LOG:  checkpoint complete: wrote 98 buffers (0.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.036 s, sync=0.165 s, total=0.275 s; sync files=102, longest=0.007 s, average=0.002 s; distance=5505 kB, estimate=5505 kB; lsn=0/975F0B8, redo lsn=0/975F060
2025-03-20 17:03:55.840 IST [22736] LOG:  checkpoint starting: immediate force wait
2025-03-20 17:03:55.883 IST [22736] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.010 s, sync=0.007 s, total=0.044 s; sync files=4, longest=0.005 s, average=0.002 s; distance=4844 kB, estimate=5439 kB; lsn=0/9C1A390, redo lsn=0/9C1A338
2025-03-20 17:03:56.258 IST [20052] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-20 17:04:04.830 IST [17636] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-20 17:04:05.152 IST [18932] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-20 17:08:55.899 IST [22736] LOG:  checkpoint starting: time
2025-03-20 17:09:03.920 IST [22736] LOG:  checkpoint complete: wrote 74 buffers (0.5%); 0 WAL file(s) added, 0 removed, 0 recycled; write=7.961 s, sync=0.029 s, total=8.022 s; sync files=29, longest=0.007 s, average=0.001 s; distance=299 kB, estimate=4925 kB; lsn=0/9C65230, redo lsn=0/9C651A0
2025-03-20 19:23:58.011 IST [22736] LOG:  checkpoint starting: time
2025-03-20 19:24:00.129 IST [22736] LOG:  checkpoint complete: wrote 20 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=2.074 s, sync=0.014 s, total=2.121 s; sync files=12, longest=0.006 s, average=0.002 s; distance=2 kB, estimate=4433 kB; lsn=0/9C65CB0, redo lsn=0/9C65C58
2025-03-20 19:28:58.184 IST [22736] LOG:  checkpoint starting: time
2025-03-20 19:28:59.538 IST [22736] LOG:  checkpoint complete: wrote 13 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.321 s, sync=0.012 s, total=1.355 s; sync files=9, longest=0.006 s, average=0.002 s; distance=1 kB, estimate=3990 kB; lsn=0/9C66390, redo lsn=0/9C66338
2025-03-20 21:52:33.627 IST [22716] LOG:  server process (PID 25424) was terminated by exception 0xC000013A
2025-03-20 21:52:33.627 IST [22716] DETAIL:  Failed process was running: SELECT "public"."Post"."id", "public"."Post"."title", "public"."Post"."content", "public"."Post"."createdAt", "public"."Post"."updatedAt", "public"."Post"."subredditId", "public"."Post"."authorId" FROM "public"."Post" WHERE "public"."Post"."subredditId" IN ($1) ORDER BY "public"."Post"."id" ASC OFFSET $2
2025-03-20 21:52:33.627 IST [22716] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-20 21:52:33.629 IST [22716] LOG:  terminating any other active server processes
2025-03-20 21:52:33.660 IST [22716] LOG:  received fast shutdown request
^C2025-03-21 10:08:54.586 IST [4628] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-21 10:08:54.650 IST [4628] LOG:  listening on IPv6 address "::1", port 5432
2025-03-21 10:08:54.650 IST [4628] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-21 10:08:54.822 IST [4772] LOG:  database system was interrupted; last known up at 2025-03-20 19:28:59 IST
2025-03-21 10:09:04.838 IST [4772] LOG:  syncing data directory (fsync), elapsed time: 10.01 s, current path: ./global/2676
2025-03-21 10:09:05.245 IST [4772] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-21 10:09:05.266 IST [4772] LOG:  redo starts at 0/9C66338
2025-03-21 10:09:05.266 IST [4772] LOG:  invalid record length at 0/9C66440: expected at least 24, got 0
2025-03-21 10:09:05.266 IST [4772] LOG:  redo done at 0/9C66408 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-21 10:09:05.290 IST [17072] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-21 10:09:05.369 IST [17072] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.026 s, sync=0.018 s, total=0.091 s; sync files=2, longest=0.010 s, average=0.009 s; distance=0 kB, estimate=0 kB; lsn=0/9C66440, redo lsn=0/9C66440
2025-03-21 10:09:05.454 IST [4628] LOG:  database system is ready to accept connections
2025-03-21 18:19:05.334 IST [4628] LOG:  server process (PID 28288) was terminated by exception 0xC000013A
2025-03-21 18:19:05.334 IST [4628] DETAIL:  Failed process was running: SELECT "public"."User"."id", "public"."User"."name", "public"."User"."username", "public"."User"."email", "public"."User"."emailVerified", "public"."User"."passwordHash", "public"."User"."image" FROM "public"."User" WHERE "public"."User"."id" IN ($1) OFFSET $2
2025-03-21 18:19:05.334 IST [4628] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-21 18:19:05.335 IST [4628] LOG:  terminating any other active server processes
2025-03-21 18:19:05.373 IST [4628] LOG:  received fast shutdown request
^C2025-03-22 11:48:09.100 IST [27108] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-22 11:48:09.114 IST [27108] LOG:  listening on IPv6 address "::1", port 5432
2025-03-22 11:48:09.114 IST [27108] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-22 11:48:09.288 IST [28372] LOG:  database system was interrupted; last known up at 2025-03-21 10:09:05 IST
2025-03-22 11:48:17.893 IST [28372] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-22 11:48:17.909 IST [28372] LOG:  redo starts at 0/9C664B8
2025-03-22 11:48:17.909 IST [28372] LOG:  invalid record length at 0/9C664F0: expected at least 24, got 0
2025-03-22 11:48:17.909 IST [28372] LOG:  redo done at 0/9C664B8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-22 11:48:17.928 IST [22656] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-22 11:48:18.001 IST [22656] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.027 s, sync=0.014 s, total=0.083 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C664F0, redo lsn=0/9C664F0
2025-03-22 11:48:18.067 IST [27108] LOG:  database system is ready to accept connections
2025-03-22 18:49:44.073 IST [21296] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-22 18:49:44.185 IST [21296] LOG:  listening on IPv6 address "::1", port 5432
2025-03-22 18:49:44.185 IST [21296] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-22 18:49:44.502 IST [20680] LOG:  database system was interrupted; last known up at 2025-03-22 11:48:18 IST
2025-03-22 18:49:54.550 IST [20680] LOG:  syncing data directory (fsync), elapsed time: 10.03 s, current path: ./base/16387/2834
2025-03-22 18:50:04.523 IST [20680] LOG:  syncing data directory (fsync), elapsed time: 20.00 s, current path: ./global/6246
2025-03-22 18:50:04.758 IST [20680] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-22 18:50:04.788 IST [20680] LOG:  redo starts at 0/9C66568
2025-03-22 18:50:04.788 IST [20680] LOG:  invalid record length at 0/9C665A0: expected at least 24, got 0
2025-03-22 18:50:04.789 IST [20680] LOG:  redo done at 0/9C66568 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-22 18:50:04.820 IST [21192] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-22 18:50:04.955 IST [21192] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.047 s, sync=0.036 s, total=0.152 s; sync files=2, longest=0.026 s, average=0.018 s; distance=0 kB, estimate=0 kB; lsn=0/9C665A0, redo lsn=0/9C665A0
2025-03-22 18:50:05.093 IST [21296] LOG:  database system is ready to accept connections
2025-03-22 20:42:19.641 IST [21296] LOG:  server process (PID 5104) was terminated by exception 0x40010004
2025-03-22 20:42:19.641 IST [21296] DETAIL:  Failed process was running: SELECT "public"."User"."id", "public"."User"."name", "public"."User"."username", "public"."User"."email", "public"."User"."emailVerified", "public"."User"."passwordHash", "public"."User"."image" FROM "public"."User" WHERE "public"."User"."id" IN ($1) OFFSET $2
2025-03-22 20:42:19.641 IST [21296] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-22 20:42:19.642 IST [21296] LOG:  terminating any other active server processes
2025-03-22 20:42:19.786 IST [21296] LOG:  all server processes terminated; reinitializing
2025-03-22 22:22:39.710 IST [4876] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-22 22:22:39.721 IST [4876] LOG:  listening on IPv6 address "::1", port 5432
2025-03-22 22:22:39.721 IST [4876] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-22 22:22:39.966 IST [15664] LOG:  database system was interrupted; last known up at 2025-03-22 18:50:04 IST
2025-03-22 22:22:48.435 IST [15664] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-22 22:22:48.452 IST [15664] LOG:  redo starts at 0/9C66618
2025-03-22 22:22:48.452 IST [15664] LOG:  invalid record length at 0/9C66650: expected at least 24, got 0
2025-03-22 22:22:48.453 IST [15664] LOG:  redo done at 0/9C66618 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-22 22:22:48.471 IST [15624] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-22 22:22:48.535 IST [15624] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.023 s, sync=0.015 s, total=0.073 s; sync files=2, longest=0.008 s, average=0.008 s; distance=0 kB, estimate=0 kB; lsn=0/9C66650, redo lsn=0/9C66650
2025-03-22 22:22:48.609 IST [4876] LOG:  database system is ready to accept connections
2025-03-22 23:28:55.270 IST [4876] LOG:  background worker "logical replication launcher" (PID 8096) was terminated by exception 0x40010004
2025-03-22 23:28:55.270 IST [4876] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-22 23:28:55.275 IST [4876] LOG:  terminating any other active server processes
^C2025-03-23 12:15:20.989 IST [3636] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-23 12:15:21.064 IST [3636] LOG:  listening on IPv6 address "::1", port 5432
2025-03-23 12:15:21.065 IST [3636] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-23 12:15:21.241 IST [3504] LOG:  database system was interrupted; last known up at 2025-03-22 22:22:48 IST
2025-03-23 12:15:28.833 IST [3504] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-23 12:15:28.848 IST [3504] LOG:  redo starts at 0/9C666C8
2025-03-23 12:15:28.848 IST [3504] LOG:  invalid record length at 0/9C66700: expected at least 24, got 0
2025-03-23 12:15:28.848 IST [3504] LOG:  redo done at 0/9C666C8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-23 12:15:28.868 IST [3208] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-23 12:15:28.930 IST [3208] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.024 s, sync=0.013 s, total=0.073 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C66700, redo lsn=0/9C66700
2025-03-23 12:15:29.002 IST [3636] LOG:  database system is ready to accept connections
2025-03-23 16:37:38.291 IST [7444] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-23 17:08:32.082 IST [3208] LOG:  checkpoint starting: time
2025-03-23 17:08:32.940 IST [3208] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.781 s, sync=0.026 s, total=0.864 s; sync files=8, longest=0.012 s, average=0.004 s; distance=1 kB, estimate=1 kB; lsn=0/9C66EE8, redo lsn=0/9C66E90
2025-03-23 17:16:24.519 IST [15020] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-23 21:54:23.303 IST [3636] LOG:  server process (PID 20040) was terminated by exception 0xC000013A
2025-03-23 21:54:23.303 IST [3636] DETAIL:  Failed process was running: SELECT "public"."User"."id", "public"."User"."name", "public"."User"."username", "public"."User"."email", "public"."User"."emailVerified", "public"."User"."passwordHash", "public"."User"."image" FROM "public"."User" WHERE "public"."User"."id" IN ($1) OFFSET $2
2025-03-23 21:54:23.303 IST [3636] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-23 21:54:23.308 IST [3636] LOG:  terminating any other active server processes
2025-03-23 21:54:23.440 IST [3636] LOG:  all server processes terminated; reinitializing
2025-03-23 21:54:23.756 IST [3636] LOG:  startup process (PID 7976) was terminated by exception 0xC0000142
2025-03-23 21:54:23.756 IST [3636] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-23 21:54:23.757 IST [3636] LOG:  aborting startup due to startup process failure
2025-03-23 21:54:23.772 IST [3636] LOG:  database system is shut down
2025-03-24 08:29:47.409 IST [7076] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-24 08:29:47.470 IST [7076] LOG:  listening on IPv6 address "::1", port 5432
2025-03-24 08:29:47.470 IST [7076] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-24 08:29:47.616 IST [12744] LOG:  database system was interrupted; last known up at 2025-03-23 17:08:32 IST
2025-03-24 08:29:54.380 IST [12744] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-24 08:29:54.390 IST [12744] LOG:  redo starts at 0/9C66E90
2025-03-24 08:29:54.390 IST [12744] LOG:  invalid record length at 0/9C66F98: expected at least 24, got 0
2025-03-24 08:29:54.391 IST [12744] LOG:  redo done at 0/9C66F60 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-24 08:29:54.404 IST [23052] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-24 08:29:54.450 IST [23052] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.016 s, sync=0.010 s, total=0.054 s; sync files=2, longest=0.006 s, average=0.005 s; distance=0 kB, estimate=0 kB; lsn=0/9C66F98, redo lsn=0/9C66F98
2025-03-24 08:29:54.510 IST [7076] LOG:  database system is ready to accept connections
2025-03-24 11:24:54.318 IST [23052] LOG:  checkpoint starting: time
2025-03-24 11:24:55.905 IST [23052] LOG:  checkpoint complete: wrote 15 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.539 s, sync=0.017 s, total=1.592 s; sync files=15, longest=0.006 s, average=0.002 s; distance=4 kB, estimate=4 kB; lsn=0/9C680D0, redo lsn=0/9C68078
2025-03-25 09:37:05.940 IST [5732] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-25 09:37:06.187 IST [5732] LOG:  listening on IPv6 address "::1", port 5432
2025-03-25 09:37:06.188 IST [5732] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-25 09:37:06.709 IST [9940] LOG:  database system was interrupted; last known up at 2025-03-24 11:24:55 IST
2025-03-25 09:37:16.533 IST [9940] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-25 09:37:16.549 IST [9940] LOG:  redo starts at 0/9C68078
2025-03-25 09:37:16.549 IST [9940] LOG:  invalid record length at 0/9C68180: expected at least 24, got 0
2025-03-25 09:37:16.549 IST [9940] LOG:  redo done at 0/9C68148 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-25 09:37:16.566 IST [23172] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-25 09:37:16.624 IST [23172] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.020 s, sync=0.011 s, total=0.067 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/9C68180, redo lsn=0/9C68180
2025-03-25 09:37:16.711 IST [5732] LOG:  database system is ready to accept connections
2025-03-25 12:01:21.040 IST [5732] LOG:  background worker "logical replication launcher" (PID 5664) was terminated by exception 0xC000013A
2025-03-25 12:01:21.040 IST [5732] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-25 12:01:21.042 IST [5732] LOG:  terminating any other active server processes
2025-03-25 12:01:21.077 IST [5732] LOG:  all server processes terminated; reinitializing
^C2025-03-25 20:23:23.324 IST [23048] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-25 20:23:23.395 IST [23048] LOG:  listening on IPv6 address "::1", port 5432
2025-03-25 20:23:23.395 IST [23048] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-25 20:23:23.554 IST [18012] LOG:  database system was interrupted; last known up at 2025-03-25 09:37:16 IST
2025-03-25 20:23:31.643 IST [18012] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-25 20:23:31.663 IST [18012] LOG:  redo starts at 0/9C681F8
2025-03-25 20:23:31.663 IST [18012] LOG:  invalid record length at 0/9C68230: expected at least 24, got 0
2025-03-25 20:23:31.663 IST [18012] LOG:  redo done at 0/9C681F8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-25 20:23:31.686 IST [23064] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-25 20:23:31.764 IST [23064] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.027 s, sync=0.016 s, total=0.090 s; sync files=2, longest=0.008 s, average=0.008 s; distance=0 kB, estimate=0 kB; lsn=0/9C68230, redo lsn=0/9C68230
2025-03-25 20:23:31.850 IST [23048] LOG:  database system is ready to accept connections
2025-03-26 12:59:29.101 IST [21380] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-26 12:59:29.198 IST [21380] LOG:  listening on IPv6 address "::1", port 5432
2025-03-26 12:59:29.198 IST [21380] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-26 12:59:29.438 IST [21432] LOG:  database system was interrupted; last known up at 2025-03-25 20:23:31 IST
2025-03-26 12:59:39.472 IST [21432] LOG:  syncing data directory (fsync), elapsed time: 10.02 s, current path: ./base/5/2687
2025-03-26 12:59:40.948 IST [21432] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-26 12:59:40.967 IST [21432] LOG:  redo starts at 0/9C682A8
2025-03-26 12:59:40.967 IST [21432] LOG:  invalid record length at 0/9C682E0: expected at least 24, got 0
2025-03-26 12:59:40.967 IST [21432] LOG:  redo done at 0/9C682A8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-26 12:59:40.988 IST [16128] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-26 12:59:41.082 IST [16128] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.026 s, sync=0.031 s, total=0.104 s; sync files=2, longest=0.016 s, average=0.016 s; distance=0 kB, estimate=0 kB; lsn=0/9C682E0, redo lsn=0/9C682E0
2025-03-26 12:59:41.200 IST [21380] LOG:  database system is ready to accept connections
2025-03-26 22:11:54.280 IST [18716] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-26 22:12:02.366 IST [21380] LOG:  received fast shutdown request
2025-03-26 22:12:02.367 IST [5544] ERROR:  canceling statement due to user request
2025-03-26 22:12:02.372 IST [21380] LOG:  aborting any active transactions
2025-03-26 22:12:02.382 IST [21380] LOG:  background worker "logical replication launcher" (PID 5544) exited with exit code 1
2025-03-26 22:12:02.392 IST [16128] LOG:  shutting down
2025-03-26 22:12:02.402 IST [16128] LOG:  checkpoint starting: shutdown immediate
2025-03-26 22:12:02.437 IST [16128] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.005 s, sync=0.001 s, total=0.045 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=0 kB; lsn=0/9C68390, redo lsn=0/9C68390
2025-03-26 22:12:02.459 IST [21380] LOG:  database system is shut down
2025-03-27 17:19:32.241 IST [20924] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-27 17:19:32.256 IST [20924] LOG:  listening on IPv6 address "::1", port 5432
2025-03-27 17:19:32.256 IST [20924] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-27 17:19:32.551 IST [21444] LOG:  database system was shut down at 2025-03-26 22:12:02 IST
2025-03-27 17:19:33.040 IST [20924] LOG:  database system is ready to accept connections
2025-03-27 17:24:32.445 IST [21412] LOG:  checkpoint starting: time
2025-03-27 17:24:34.083 IST [21412] LOG:  checkpoint complete: wrote 17 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.561 s, sync=0.032 s, total=1.644 s; sync files=16, longest=0.009 s, average=0.002 s; distance=6 kB, estimate=6 kB; lsn=0/9C69D08, redo lsn=0/9C69CB0
2025-03-27 19:28:35.552 IST [12724] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-27 19:28:35.557 IST [14476] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-28 02:43:46.476 IST [20924] LOG:  server process (PID 24892) was terminated by exception 0x40010004
2025-03-28 02:43:46.476 IST [20924] DETAIL:  Failed process was running: SELECT "public"."Subreddit"."id", "public"."Subreddit"."name", "public"."Subreddit"."createdAt", "public"."Subreddit"."updatedAt", "public"."Subreddit"."creatorId", "public"."Subreddit"."description", "public"."Subreddit"."image", "public"."Subreddit"."backgroundImage" FROM "public"."Subreddit" WHERE "public"."Subreddit"."name" = $1 LIMIT $2 OFFSET $3
2025-03-28 02:43:46.476 IST [20924] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-28 02:43:46.477 IST [20924] LOG:  terminating any other active server processes
2025-03-28 02:43:46.624 IST [20924] LOG:  all server processes terminated; reinitializing
2025-03-28 02:43:47.098 IST [20924] LOG:  startup process (PID 25352) was terminated by exception 0xC0000142
2025-03-28 02:43:47.098 IST [20924] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-28 02:43:47.098 IST [20924] LOG:  aborting startup due to startup process failure
2025-03-28 02:43:47.105 IST [20924] LOG:  database system is shut down
2025-03-28 02:54:58.874 IST [6080] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-28 02:54:59.011 IST [6080] LOG:  listening on IPv6 address "::1", port 5432
2025-03-28 02:54:59.012 IST [6080] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-28 02:54:59.396 IST [10732] LOG:  database system was interrupted; last known up at 2025-03-27 17:24:34 IST
2025-03-28 02:55:09.425 IST [10732] LOG:  syncing data directory (fsync), elapsed time: 10.01 s, current path: ./base/4/14897_vm
2025-03-28 02:55:14.304 IST [10732] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-28 02:55:14.324 IST [10732] LOG:  redo starts at 0/9C69CB0
2025-03-28 02:55:14.324 IST [10732] LOG:  invalid record length at 0/9C69DB8: expected at least 24, got 0
2025-03-28 02:55:14.324 IST [10732] LOG:  redo done at 0/9C69D80 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-28 02:55:14.345 IST [6700] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-28 02:55:14.411 IST [6700] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.023 s, sync=0.014 s, total=0.076 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C69DB8, redo lsn=0/9C69DB8
2025-03-28 02:55:14.561 IST [6080] LOG:  database system is ready to accept connections
2025-03-28 03:03:27.840 IST [15396] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-28 21:48:03.688 IST [8392] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-28 21:48:03.761 IST [8392] LOG:  listening on IPv6 address "::1", port 5432
2025-03-28 21:48:03.761 IST [8392] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-28 21:48:03.926 IST [19788] LOG:  database system was interrupted; last known up at 2025-03-28 02:55:14 IST
2025-03-28 21:48:12.691 IST [19788] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-28 21:48:12.708 IST [19788] LOG:  redo starts at 0/9C69E30
2025-03-28 21:48:12.708 IST [19788] LOG:  invalid record length at 0/9C69E68: expected at least 24, got 0
2025-03-28 21:48:12.708 IST [19788] LOG:  redo done at 0/9C69E30 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-28 21:48:12.729 IST [24840] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-28 21:48:12.803 IST [24840] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.025 s, sync=0.015 s, total=0.084 s; sync files=2, longest=0.008 s, average=0.008 s; distance=0 kB, estimate=0 kB; lsn=0/9C69E68, redo lsn=0/9C69E68
2025-03-28 21:48:12.883 IST [8392] LOG:  database system is ready to accept connections
2025-03-28 23:48:02.391 IST [8392] LOG:  received fast shutdown request
2025-03-28 23:48:02.400 IST [8392] LOG:  aborting any active transactions
2025-03-28 23:48:02.401 IST [6736] ERROR:  canceling statement due to user request
2025-03-28 23:48:02.401 IST [8636] ERROR:  canceling statement due to user request
2025-03-28 23:48:02.423 IST [8392] LOG:  background worker "logical replication launcher" (PID 6736) exited with exit code 1
2025-03-28 23:48:02.424 IST [24840] LOG:  shutting down
2025-03-28 23:48:02.438 IST [24840] LOG:  checkpoint starting: shutdown immediate
2025-03-28 23:48:02.475 IST [24840] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.007 s, sync=0.001 s, total=0.051 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=0 kB; lsn=0/9C69F18, redo lsn=0/9C69F18
2025-03-28 23:48:02.506 IST [8392] LOG:  database system is shut down
2025-03-29 05:37:31.146 IST [12552] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-29 05:37:31.174 IST [12552] LOG:  listening on IPv6 address "::1", port 5432
2025-03-29 05:37:31.174 IST [12552] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-29 05:37:31.459 IST [4800] LOG:  database system was shut down at 2025-03-28 23:48:02 IST
2025-03-29 05:37:31.771 IST [12552] LOG:  database system is ready to accept connections
2025-03-29 05:42:31.406 IST [19588] LOG:  checkpoint starting: time
2025-03-29 05:42:31.493 IST [19588] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.038 s, sync=0.013 s, total=0.094 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A038, redo lsn=0/9C69FC8
2025-03-29 16:52:07.210 IST [21128] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-29 16:52:07.281 IST [21128] LOG:  listening on IPv6 address "::1", port 5432
2025-03-29 16:52:07.281 IST [21128] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-29 16:52:07.446 IST [3640] LOG:  database system was interrupted; last known up at 2025-03-29 05:42:31 IST
2025-03-29 16:52:16.187 IST [3640] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-29 16:52:16.205 IST [3640] LOG:  redo starts at 0/9C69FC8
2025-03-29 16:52:16.205 IST [3640] LOG:  invalid record length at 0/9C6A0E8: expected at least 24, got 0
2025-03-29 16:52:16.205 IST [3640] LOG:  redo done at 0/9C6A0B0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-29 16:52:16.226 IST [20476] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-29 16:52:16.317 IST [20476] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.038 s, sync=0.019 s, total=0.102 s; sync files=2, longest=0.010 s, average=0.010 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A0E8, redo lsn=0/9C6A0E8
2025-03-29 16:52:16.475 IST [21128] LOG:  database system is ready to accept connections
2025-03-30 17:25:15.043 IST [15384] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-30 17:25:15.117 IST [15384] LOG:  listening on IPv6 address "::1", port 5432
2025-03-30 17:25:15.117 IST [15384] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-30 17:25:15.277 IST [2472] LOG:  database system was interrupted; last known up at 2025-03-29 16:52:16 IST
2025-03-30 17:25:23.938 IST [2472] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-30 17:25:23.954 IST [2472] LOG:  redo starts at 0/9C6A160
2025-03-30 17:25:23.955 IST [2472] LOG:  invalid record length at 0/9C6A198: expected at least 24, got 0
2025-03-30 17:25:23.955 IST [2472] LOG:  redo done at 0/9C6A160 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-30 17:25:23.977 IST [8684] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-30 17:25:24.053 IST [8684] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.028 s, sync=0.015 s, total=0.087 s; sync files=2, longest=0.008 s, average=0.008 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A198, redo lsn=0/9C6A198
2025-03-30 17:25:24.143 IST [15384] LOG:  database system is ready to accept connections
2025-03-30 19:45:34.421 IST [14720] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-31 11:21:38.482 IST [4568] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-03-31 11:21:38.609 IST [4568] LOG:  listening on IPv6 address "::1", port 5432
2025-03-31 11:21:38.609 IST [4568] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-03-31 11:21:38.826 IST [24088] LOG:  database system was interrupted; last known up at 2025-03-30 17:25:24 IST
2025-03-31 11:21:48.837 IST [24088] LOG:  syncing data directory (fsync), elapsed time: 10.00 s, current path: ./base/5/6239
2025-03-31 11:21:49.475 IST [24088] LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-31 11:21:49.497 IST [24088] LOG:  redo starts at 0/9C6A210
2025-03-31 11:21:49.497 IST [24088] LOG:  invalid record length at 0/9C6A248: expected at least 24, got 0
2025-03-31 11:21:49.497 IST [24088] LOG:  redo done at 0/9C6A210 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-03-31 11:21:49.528 IST [23896] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-31 11:21:49.631 IST [23896] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.036 s, sync=0.020 s, total=0.117 s; sync files=2, longest=0.011 s, average=0.010 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A248, redo lsn=0/9C6A248
2025-03-31 11:21:49.760 IST [4568] LOG:  database system is ready to accept connections
2025-03-31 13:21:44.789 IST [11228] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-31 15:05:09.169 IST [2452] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-03-31 15:05:09.354 IST [4568] LOG:  background worker "logical replication launcher" (PID 2468) was terminated by exception 0xC000013A
2025-03-31 15:05:09.354 IST [4568] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-31 15:05:09.354 IST [4568] LOG:  terminating any other active server processes
2025-03-31 15:05:09.379 IST [4568] LOG:  all server processes terminated; reinitializing
2025-03-31 15:05:09.474 IST [4568] LOG:  startup process (PID 24004) was terminated by exception 0xC0000142
2025-03-31 15:05:09.474 IST [4568] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-03-31 15:05:09.474 IST [4568] LOG:  aborting startup due to startup process failure
2025-03-31 15:05:09.479 IST [4568] LOG:  database system is shut down
2025-04-01 09:42:20.607 IST [7352] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-01 09:42:20.619 IST [7352] LOG:  listening on IPv6 address "::1", port 5432
2025-04-01 09:42:20.619 IST [7352] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-01 09:42:20.784 IST [21876] LOG:  database system was interrupted; last known up at 2025-03-31 11:21:49 IST
2025-04-01 09:42:27.792 IST [21876] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-01 09:42:27.808 IST [21876] LOG:  redo starts at 0/9C6A2C0
2025-04-01 09:42:27.808 IST [21876] LOG:  invalid record length at 0/9C6A2F8: expected at least 24, got 0
2025-04-01 09:42:27.808 IST [21876] LOG:  redo done at 0/9C6A2C0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-01 09:42:27.825 IST [22952] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-01 09:42:27.883 IST [22952] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.023 s, sync=0.012 s, total=0.069 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A2F8, redo lsn=0/9C6A2F8
2025-04-01 09:42:27.943 IST [7352] LOG:  database system is ready to accept connections
2025-04-01 19:48:11.000 IST [20324] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-01 19:48:11.069 IST [20324] LOG:  listening on IPv6 address "::1", port 5432
2025-04-01 19:48:11.069 IST [20324] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-01 19:48:11.221 IST [5288] LOG:  database system was interrupted; last known up at 2025-04-01 09:42:27 IST
2025-04-01 19:48:19.391 IST [5288] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-01 19:48:19.408 IST [5288] LOG:  redo starts at 0/9C6A370
2025-04-01 19:48:19.409 IST [5288] LOG:  invalid record length at 0/9C6A3A8: expected at least 24, got 0
2025-04-01 19:48:19.409 IST [5288] LOG:  redo done at 0/9C6A370 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-01 19:48:19.426 IST [5184] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-01 19:48:19.488 IST [5184] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.024 s, sync=0.013 s, total=0.071 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A3A8, redo lsn=0/9C6A3A8
2025-04-01 19:48:19.572 IST [20324] LOG:  database system is ready to accept connections
2025-04-02 08:37:42.289 IST [16336] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-02 08:37:42.360 IST [16336] LOG:  listening on IPv6 address "::1", port 5432
2025-04-02 08:37:42.360 IST [16336] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-02 08:37:42.528 IST [23188] LOG:  database system was interrupted; last known up at 2025-04-01 19:48:19 IST
2025-04-02 08:37:49.261 IST [23188] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-02 08:37:49.275 IST [23188] LOG:  redo starts at 0/9C6A420
2025-04-02 08:37:49.275 IST [23188] LOG:  invalid record length at 0/9C6A458: expected at least 24, got 0
2025-04-02 08:37:49.275 IST [23188] LOG:  redo done at 0/9C6A420 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-02 08:37:49.291 IST [17240] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-02 08:37:49.347 IST [17240] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.019 s, sync=0.014 s, total=0.064 s; sync files=2, longest=0.008 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A458, redo lsn=0/9C6A458
2025-04-02 08:37:49.413 IST [16336] LOG:  database system is ready to accept connections
2025-04-02 10:07:42.228 IST [22308] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-02 10:20:28.705 IST [4112] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-02 19:28:44.244 IST [21328] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-02 19:28:44.319 IST [21328] LOG:  listening on IPv6 address "::1", port 5432
2025-04-02 19:28:44.319 IST [21328] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-02 19:28:44.592 IST [26132] LOG:  database system was interrupted; last known up at 2025-04-02 08:37:49 IST
2025-04-02 19:28:53.117 IST [26132] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-02 19:28:53.133 IST [26132] LOG:  redo starts at 0/9C6A4D0
2025-04-02 19:28:53.133 IST [26132] LOG:  invalid record length at 0/9C6A508: expected at least 24, got 0
2025-04-02 19:28:53.134 IST [26132] LOG:  redo done at 0/9C6A4D0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-02 19:28:53.152 IST [2216] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-02 19:28:53.221 IST [2216] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.025 s, sync=0.015 s, total=0.078 s; sync files=2, longest=0.008 s, average=0.008 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A508, redo lsn=0/9C6A508
2025-04-02 19:28:53.296 IST [21328] LOG:  database system is ready to accept connections
2025-04-04 08:53:37.483 IST [19396] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-04 08:53:37.702 IST [19396] LOG:  listening on IPv6 address "::1", port 5432
2025-04-04 08:53:37.703 IST [19396] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-04 08:53:38.253 IST [13324] LOG:  database system was interrupted; last known up at 2025-04-02 19:28:53 IST
2025-04-04 08:53:48.373 IST [13324] LOG:  syncing data directory (fsync), elapsed time: 10.03 s, current path: ./base/16387/57699
2025-04-04 08:53:55.082 IST [13324] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-04 08:53:55.103 IST [13324] LOG:  redo starts at 0/9C6A580
2025-04-04 08:53:55.104 IST [13324] LOG:  invalid record length at 0/9C6A5B8: expected at least 24, got 0
2025-04-04 08:53:55.104 IST [13324] LOG:  redo done at 0/9C6A580 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-04 08:53:55.127 IST [15512] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-04 08:53:55.214 IST [15512] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.035 s, sync=0.017 s, total=0.097 s; sync files=2, longest=0.010 s, average=0.009 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A5B8, redo lsn=0/9C6A5B8
2025-04-04 08:53:55.317 IST [19396] LOG:  database system is ready to accept connections
2025-04-04 10:21:41.303 IST [11136] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-05 12:09:46.118 IST [21624] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-05 12:09:46.130 IST [21624] LOG:  listening on IPv6 address "::1", port 5432
2025-04-05 12:09:46.131 IST [21624] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-05 12:09:46.313 IST [21736] LOG:  database system was interrupted; last known up at 2025-04-04 08:53:55 IST
2025-04-05 12:09:56.096 IST [21736] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-05 12:09:56.114 IST [21736] LOG:  redo starts at 0/9C6A630
2025-04-05 12:09:56.114 IST [21736] LOG:  invalid record length at 0/9C6A668: expected at least 24, got 0
2025-04-05 12:09:56.115 IST [21736] LOG:  redo done at 0/9C6A630 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-05 12:09:56.133 IST [21704] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-05 12:09:56.197 IST [21704] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.023 s, sync=0.013 s, total=0.073 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C6A668, redo lsn=0/9C6A668
2025-04-05 12:09:56.347 IST [21624] LOG:  database system is ready to accept connections
2025-04-05 12:19:56.212 IST [21704] LOG:  checkpoint starting: time
2025-04-05 12:19:56.396 IST [21704] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.122 s, sync=0.017 s, total=0.184 s; sync files=2, longest=0.009 s, average=0.009 s; distance=1 kB, estimate=1 kB; lsn=0/9C6AAC0, redo lsn=0/9C6AA68
2025-04-05 18:51:57.229 IST [1868] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-05 19:00:44.436 IST [21624] LOG:  background worker "logical replication launcher" (PID 20964) was terminated by exception 0xC000013A
2025-04-05 19:00:44.436 IST [21624] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-04-05 19:00:44.436 IST [21624] LOG:  terminating any other active server processes
2025-04-05 19:00:44.450 IST [21624] LOG:  received fast shutdown request
^C2025-04-05 19:05:16.727 IST [2852] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-05 19:05:16.745 IST [2852] LOG:  listening on IPv6 address "::1", port 5432
2025-04-05 19:05:16.745 IST [2852] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-05 19:05:16.964 IST [18172] LOG:  database system was interrupted; last known up at 2025-04-05 12:19:56 IST
2025-04-05 19:05:24.824 IST [18172] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-05 19:05:24.840 IST [18172] LOG:  redo starts at 0/9C6AA68
2025-04-05 19:05:24.840 IST [18172] LOG:  invalid record length at 0/9C6AB70: expected at least 24, got 0
2025-04-05 19:05:24.840 IST [18172] LOG:  redo done at 0/9C6AB38 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-05 19:05:24.858 IST [3040] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-05 19:05:24.926 IST [3040] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.026 s, sync=0.013 s, total=0.077 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C6AB70, redo lsn=0/9C6AB70
2025-04-05 19:05:25.022 IST [2852] LOG:  database system is ready to accept connections
2025-04-05 19:07:23.383 IST [1320] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-05 19:26:43.123 IST [2852] LOG:  server process (PID 6492) was terminated by exception 0xC000013A
2025-04-05 19:26:43.123 IST [2852] DETAIL:  Failed process was running: SELECT "public"."User"."id", "public"."User"."name", "public"."User"."username", "public"."User"."email", "public"."User"."emailVerified", "public"."User"."passwordHash", "public"."User"."image" FROM "public"."User" WHERE "public"."User"."id" IN ($1) OFFSET $2
2025-04-05 19:26:43.123 IST [2852] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-04-05 19:26:43.124 IST [2852] LOG:  terminating any other active server processes
2025-04-05 19:26:43.159 IST [2852] LOG:  received fast shutdown request
^C2025-04-06 11:40:16.025 IST [4872] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-06 11:40:16.094 IST [4872] LOG:  listening on IPv6 address "::1", port 5432
2025-04-06 11:40:16.094 IST [4872] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-06 11:40:16.259 IST [8768] LOG:  database system was interrupted; last known up at 2025-04-05 19:05:24 IST
2025-04-06 11:40:24.484 IST [8768] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-06 11:40:24.503 IST [8768] LOG:  redo starts at 0/9C6ABE8
2025-04-06 11:40:24.503 IST [8768] LOG:  invalid record length at 0/9C6AC20: expected at least 24, got 0
2025-04-06 11:40:24.503 IST [8768] LOG:  redo done at 0/9C6ABE8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-06 11:40:24.522 IST [2528] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-06 11:40:24.584 IST [2528] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.023 s, sync=0.013 s, total=0.071 s; sync files=2, longest=0.007 s, average=0.007 s; distance=0 kB, estimate=0 kB; lsn=0/9C6AC20, redo lsn=0/9C6AC20
2025-04-06 11:40:24.657 IST [4872] LOG:  database system is ready to accept connections
2025-04-06 12:33:09.873 IST [23308] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-06 19:04:02.697 IST [21180] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-06 19:36:56.236 IST [2588] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-06 20:14:45.715 IST [11908] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-06 20:21:54.893 IST [24140] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-06 20:21:55.276 IST [4872] LOG:  background worker "logical replication launcher" (PID 1368) was terminated by exception 0xC000013A
2025-04-06 20:21:55.276 IST [4872] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-04-06 20:21:55.276 IST [4872] LOG:  terminating any other active server processes
2025-04-06 20:21:55.309 IST [4872] LOG:  received fast shutdown request
^C2025-04-07 09:31:59.448 IST [9880] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-07 09:31:59.516 IST [9880] LOG:  listening on IPv6 address "::1", port 5432
2025-04-07 09:31:59.517 IST [9880] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-07 09:31:59.706 IST [23396] LOG:  database system was interrupted; last known up at 2025-04-06 11:40:24 IST
2025-04-07 09:32:06.785 IST [23396] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-07 09:32:06.799 IST [23396] LOG:  redo starts at 0/9C6AC98
2025-04-07 09:32:06.799 IST [23396] LOG:  invalid record length at 0/9C6ACD0: expected at least 24, got 0
2025-04-07 09:32:06.799 IST [23396] LOG:  redo done at 0/9C6AC98 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-07 09:32:06.812 IST [17228] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-07 09:32:06.862 IST [17228] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.018 s, sync=0.011 s, total=0.056 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/9C6ACD0, redo lsn=0/9C6ACD0
2025-04-07 09:32:06.945 IST [9880] LOG:  database system is ready to accept connections
2025-04-07 09:52:56.040 IST [3356] LOG:  could not receive data from client: An existing connection was forcibly closed by the remote host.
	
2025-04-07 10:12:06.823 IST [17228] LOG:  checkpoint starting: time
2025-04-07 10:12:07.759 IST [17228] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.890 s, sync=0.011 s, total=0.941 s; sync files=9, longest=0.006 s, average=0.002 s; distance=3 kB, estimate=3 kB; lsn=0/9C6B9F0, redo lsn=0/9C6B998
2025-04-07 15:47:18.205 IST [9880] LOG:  server process (PID 24352) was terminated by exception 0xC000013A
2025-04-07 15:47:18.205 IST [9880] DETAIL:  Failed process was running: SELECT "public"."User"."id", "public"."User"."name", "public"."User"."username", "public"."User"."email", "public"."User"."emailVerified", "public"."User"."passwordHash", "public"."User"."image" FROM "public"."User" WHERE "public"."User"."id" IN ($1) OFFSET $2
2025-04-07 15:47:18.205 IST [9880] HINT:  See C include file "ntstatus.h" for a description of the hexadecimal value.
2025-04-07 15:47:18.207 IST [9880] LOG:  terminating any other active server processes
2025-04-07 15:47:18.340 IST [9880] LOG:  received fast shutdown request
^C2025-04-09 17:54:11.053 IST [17120] LOG:  starting PostgreSQL 17.2 on x86_64-windows, compiled by msvc-19.42.34435, 64-bit
2025-04-09 17:54:11.120 IST [17120] LOG:  listening on IPv6 address "::1", port 5432
2025-04-09 17:54:11.120 IST [17120] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-04-09 17:54:11.269 IST [21996] LOG:  database system was interrupted; last known up at 2025-04-07 10:12:07 IST
2025-04-09 17:54:17.824 IST [21996] LOG:  database system was not properly shut down; automatic recovery in progress
2025-04-09 17:54:17.837 IST [21996] LOG:  redo starts at 0/9C6B998
2025-04-09 17:54:17.838 IST [21996] LOG:  invalid record length at 0/9C6BAA0: expected at least 24, got 0
2025-04-09 17:54:17.838 IST [21996] LOG:  redo done at 0/9C6BA68 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-04-09 17:54:17.852 IST [20124] LOG:  checkpoint starting: end-of-recovery immediate wait
2025-04-09 17:54:17.905 IST [20124] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.020 s, sync=0.011 s, total=0.060 s; sync files=2, longest=0.006 s, average=0.006 s; distance=0 kB, estimate=0 kB; lsn=0/9C6BAA0, redo lsn=0/9C6BAA0
2025-04-09 17:54:17.975 IST [17120] LOG:  database system is ready to accept connections
